<div class="article fulltext-view">
    <div class="section intro" id="s1">
        <h2>Introduction</h2>
        <p>The dynamics of most chronic viral infections such as HIV, hepatitis B and C, and cytomegalovirus are assessed by serially sampling blood using polymerase chain reaction (<a class="xref-bibr" href="#bib21">Perelson et al., 1996</a>), and by ex vivo quantitation of circulating lymphocytes to estimate immune response (<a class="xref-bibr" href="#bib32">Sylwester et al., 2005</a>). For disseminated infections, sampling from blood may reflect true host–pathogen dynamics due to homogeneous mixing of viruses and PBMCs, which originate from thousands of infectious foci. However, many infections are confined to small anatomic regions; localized host–pathogen interactions within specific organ tissue constitutes the predominant site of pathogenesis, and measurements of viral replication and immune response are not accurately reflected in the blood compartment. Moreover, sampling of infected tissue often reveals that the density of virus and host immune cells differ enormously across small distances. Spatial features of infection are a critical, and relatively unexplored, component of viral dynamics.</p>
        <p>Herpes simplex virus-2 (HSV-2) is useful for studying localized aspects of pathogenesis in humans. HSV-2 replicates in highly accessible genital keratinocytes. Serial genital swabbing reveals frequent, highly variable shedding episodes in most infected persons (<a class="xref-bibr" href="#bib36">Wald et al., 1995</a>, <a class="xref-bibr" href="#bib37">1997</a>, <a class="xref-bibr" href="#bib38">2000</a>; <a class="xref-bibr" href="#bib5">Crespi et al., 2007</a>; <a class="xref-bibr" href="#bib18">Mark et al., 2008</a>), and biopsies show dense clusters of CD4+ and CD8+ lymphocytes in focal areas of viral replication (<a class="xref-bibr" href="#bib40">Zhu et al., 2007</a>). Some episodes are associated with lesions, and prolonged viral production, while others are asymptomatic and brief (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>). Markedly different episodes occur within 5–10 days of each other, and viral levels fluctuate dramatically even within a single episode (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>; <a class="xref-bibr" href="#bib28">Schiffer et al., 2011</a>). At least 20% of episodes are notable for complex erratic viral trajectories, including re-expansion phases that represent failure of the immune system to contain replication (<a class="xref-bibr" href="#bib28">Schiffer et al., 2011</a>). Past modeling suggests that nearly constant reactivation of HSV in a small percentage of infected ganglionic neurons might account for high frequency of shedding episodes (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>). Animal and mathematical models predict that varying T-cell responses account for episode heterogeneity (<a class="xref-bibr" href="#bib11">Gebhardt et al., 2009</a>; <a class="xref-bibr" href="#bib26">Schiffer et al., 2010</a>). However, the field lacks hypotheses to address the complex morphology of prolonged viral shedding episodes.</p>
        <p>We performed detailed studies with genital sampling performed over four different time intervals, and concurrently at multiple sites across the genital tract, to precisely define spatiotemporal kinetics of genital HSV-2 replication in immunocompetent patients. We then designed a mathematical model as a tool to develop hypotheses that explain virological data from these cohorts. Model simulations suggest very rapid rates of HSV-2 replication and epidermal cell-to-cell spread. Yet, host containment of virus within a single focus of infection occurs in &lt;24 hr, necessitating seeding of adjacent regions to promote increased shedding.</p>
    </div>
    <div class="section results" id="s2">
        <h2>Results</h2>
        <div id="s2-1" class="subsection">
            <h3>Temporal kinetics of HSV-2 genital replication</h3>
            <p>HSV-2 shedding in the genital tract is characterized by intermittent frequent episodes of variable duration (hours to weeks) and viral load (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>; <a class="xref-bibr" href="#bib28">Schiffer et al., 2011</a>). To evaluate viral patterns during individual shedding episodes, we analyzed data from three cohorts of immunocompetent patients who underwent genital tract swabbing at different time intervals (<a class="xref-table" href="#tbl1">Table 1</a>; <a class="xref-boxed-text" href="#box1">Boxes 1–3</a>). The same collection methods and HSV PCR assay were used in each cohort. Swabs were tested for HSV DNA using a validated quantitative PCR assay with a sensitivity of 1 copy/reaction (<a class="xref-bibr" href="#bib13">Jerome et al., 2002</a>; <a class="xref-bibr" href="#bib16">Magaret et al., 2007</a>). These studies demonstrated that quantity of genital HSV DNA is stable over minutes but expands and decays extremely rapidly over hours, and that prolonged episodes are notable for frequent and erratic peaks in HSV-2 viral load (<a class="xref-fig" href="#fig1">Figure 1</a>, <a class="xref-fig" href="#fig1s1">Figure 1—figure supplements 1–3</a>).</p>

            <div class="table-expansion" id="tbl1"><div class="table-caption"><span class="table-label">Table 1.</span> <p>Five cohorts of HSV-2 genital tract shedding</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.003">http://dx.doi.org/10.7554/eLife.00288.003</a></p><div class="sb-div caption-clear"></div></div><table><thead><tr><td>Cohort</td><td>Subjects</td><td>Total swabs</td><td>Swabbing frequency</td><td>Total episodes</td><td>Swabbing duration</td><td>Anatomic swabbing region</td><td>Purpose</td></tr></thead><tbody><tr><td>A</td><td>3</td><td>96</td><td>Every 5 min</td><td>3</td><td>4 hr when lesion present</td><td>Total genital tract</td><td>Swab-to-swab sampling/assay variability</td></tr><tr><td>B</td><td>5</td><td>200</td><td>10 times/day (every 2 hr during the days and 4 hr overnight)</td><td>5</td><td>4–5 days when lesion present</td><td>Total genital tract</td><td>Episode expansion, clearance and re-expansion kinetics</td></tr><tr><td>C</td><td>25</td><td>4706</td><td>4 times/day</td><td>109</td><td>30–60 days without or with a lesion</td><td>Total genital tract</td><td>Accurate estimates for expansion/decay slopes for clinical and subclinical episodes</td></tr><tr><td>D</td><td>2</td><td>216</td><td>Daily</td><td>4</td><td>30 days without or with a lesion</td><td>23 separate regions</td><td>Spatial dispersion of HSV</td></tr><tr><td>E</td><td>531</td><td>14,685</td><td>Daily</td><td>1020</td><td>&gt;30 days with or without a lesion</td><td>Total genital tract</td><td>Model fit</td></tr></tbody></table></div>
            <div class="boxed-text" id="box1"><span class="boxed-text-label">Box 1.</span><p class="first-child">Cohort A</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.004">http://dx.doi.org/10.7554/eLife.00288.004</a></p><p>Cohort A included three subjects who were seen within 12 hr of HSV-2 recurrence, and who collected genital swabs every 5 min for 4 hr. This experiment assessed whether there were alterations in viral load related to the PCR assay or collection technique. Only small changes occurred during 5-min intervals (<a class="xref-fig" href="#fig1">Figure 1A</a>, <a class="xref-fig" href="#fig1s1">Figure 1—figure supplement 1A,B</a>, <a class="xref-supplement-material" href="#SD1-data">Figure 1—source data 1</a>). The mean absolute value of viral DNA copy difference between successive swabs was 0.20, 0.28, and 0.34 log<sub>10</sub> genomic copies in the three participants, and generally increased with time between swabs (<a class="xref-fig" href="#fig1">Figure 1B</a>, <a class="xref-fig" href="#fig1s1">Figure 1—figure supplement 1C</a>). The mean difference in HSV DNA copies between swabs correlated tightly with time between swabs in two participants (<a class="xref-fig" href="#fig1">Figure 1C</a>, <a class="xref-fig" href="#fig1s1">Figure 1—figure supplement 1D</a>), with virtually no difference in viral quantity between swabs separated by 5-min intervals in all three subjects (<a class="xref-fig" href="#fig1">Figure 1C</a>, <a class="xref-fig" href="#fig1s1">Figure 1—figure supplement 1D</a>). Two episodes had negative mean differences in viral quantity with increasing time between swabs (<a class="xref-fig" href="#fig1">Figure 1C</a> , <a class="xref-fig" href="#fig1s1">Figure 1—figure supplement 1D</a>), while one episode had a slightly positive mean difference (<a class="xref-fig" href="#fig1s1">Figure 1—figure supplement 1D</a>), suggesting that we captured two episodes during a decay phase and one episode near a peak of viral production. The standard deviation of the mean absolute value of viral DNA copy difference between successive swabs was 0.16, 0.22, and 0.26 in the three participants, respectively, suggesting that differences of &gt;0.5 log<sub>10</sub> HSV-2 DNA copies that may occur over hours usually reflect true changes in viral load rather than noise in the data attributable to clinical collection or PCR technique.</p></div>
            <div class="boxed-text" id="box2"><span class="boxed-text-label">Box 2.</span><p class="first-child">Cohort B</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.005">http://dx.doi.org/10.7554/eLife.00288.005</a></p><p>To explore patterns of viral expansion and decay of an individual episode over time, we expanded the duration of evaluation and time interval between sampling. For Cohort B, we enrolled five HSV-2 positive participants with genital lesions, and sampled them 10 times daily (every 2 hr during the day and 4 hr overnight) for 4–5 days from lesion onset. A key feature of all episodes was a ‘saw tooth’ morphology of viral expansion and contraction (<a class="xref-fig" href="#fig1">Figure 1D</a>, <a class="xref-fig" href="#fig1s1">Figure 1—figure supplement 2</a>, <a class="xref-supplement-material" href="#SD1-data">Figure 1—source data 1</a>), with 0.8–1.8 daily peaks (defined as an expansion of 0.5 log<sub>10</sub> HSV DNA followed by a decline of 0.5 log<sub>10</sub> HSV DNA). The mean absolute value of viral DNA copy change between successive swabs separated by 2 hr was 0.73, 0.72, 0.50, 0.51, and 0.34 log<sub>10</sub> HSV DNA copies in the five participants, respectively, suggesting significant variability in viral levels. In addition, each episode was notable for high initial viral peak (6–8 log<sub>10</sub> HSV DNA) followed by decay within 24 hr of initiation, highlighting extremely early rapid viral expansion, followed by rapid control.</p></div>
            <div class="boxed-text" id="box3"><span class="boxed-text-label">Box 3.</span><p class="first-child">Cohort C</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.006">http://dx.doi.org/10.7554/eLife.00288.006</a></p><p>To expand data available for analysis of early viral expansion and late viral clearance, we performed new analyses from a published cohort of 25 subjects who were sampled every 6 hr for 30–60 days (Cohort C) (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>). Among 109 episodes, 75% of which were subclinical, median regression slopes from initiation to peak and peak to termination were 20.3 and −8.7 log<sub>10</sub>/day, respectively. Median first and last copy numbers were 3.5 and 3.3 log<sub>10</sub> HSV DNA copies, respectively: these measures occurred a median of 3 hr after episode initiation or before termination, suggesting that rapid changes (∼10-fold/hr) in genomic copy numbers occurred both early and late within an episode (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>). While a majority of episodes in Cohort C were &lt;12 hr, even among the 55 episodes of &gt;24 hr, transition from initial expansion to clearance phase occurred at a median of 12 hr, implying early immunologic pressure against the virus. Rapid expansion and contraction phases continued throughout more persistent shedding episodes. <a class="xref-fig" href="#fig1">Figure 1E</a> and <a class="xref-fig" href="#fig1s3">Figure 1—figure supplement 3</a> (<a class="xref-supplement-material" href="#SD1-data">Figure 1—source data 1</a>) illustrate episodes of 17-day and 8-day duration from two participants with multiple sharp peaks from steep expansion and decay phases.</p></div>
            <div class="fig-inline-img-set-carousel"><div class="elife-fig-slider-wrapper"><div class="elife-fig-slider"><div class="elife-fig-slider-img elife-fig-slider-primary"><a href="#fig1"><img src="[graphic-elife00288f001-small]" alt="Figure 1."/></a></div><div class="figure-carousel-inner-wrapper"><div class="figure-carousel"><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig1s1"><img src="[graphic-elife00288fs001-small]" alt="Figure 1—figure supplement 1."/></a></div><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig1s2"><img src="[graphic-elife00288fs002-small]" alt="Figure 1—figure supplement 2."/></a></div><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig1s3"><img src="[graphic-elife00288fs003-small]" alt="Figure 1—figure supplement 3."/></a></div></div></div></div></div><div class="elife-fig-image-caption-wrapper"><div class="fig" data-doi="10.7554/eLife.00288.007"><div id="fig1" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288f001-large]" class="figure-expand-popup" title="Figure 1."><img data-img="[graphic-elife00288f001-small]" src="[graphic-elife00288f001-medium]" alt="Figure 1."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288f001-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288f001-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 1.</span> <span class="caption-title">HSV-2 levels in the genital tract are stable over minutes, expand and decay markedly over hours, and fluctuate rapidly and unpredictably over days.</span><p>(<strong>A</strong>) Shedding quantity in a participant, who performed genital swabs every 5 min over 4 hr during a lesion, reveals low swab-to-swab variation in viral quantity. Using data from panel (<strong>A</strong> and <strong>B</strong>), absolute mean difference (R<sup>2</sup> = 0.99), and (<strong>C</strong>) mean difference (R<sup>2</sup> = 0.87), in HSV DNA copies between swabs, are a function of time between swabs. (<strong>D</strong>) Shedding quantity in a participant, who performed 10 genital swabs per day during a lesion over 4 days (swabs every 2–4 hr), shows a characteristic saw-tooth pattern; arrows denote rapid viral re-expansion; the participant had a negative swab performed before episode onset. (<strong>E</strong>) Shedding quantity in a participant, who performed four genital swabs per day over 17 days demonstrates that rapid and frequent viral re-expansion allows for shedding prolongation; four missing data points are left blank.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.007">http://dx.doi.org/10.7554/eLife.00288.007</a></p><div class="supplementary-material" data-doi="10.7554/eLife.00288.008"><div class="supplementary-material-expansion" id="SD1-data"><span class="supplementary-material-label">Figure 1—source data 1.</span><p>Source data for Figure 1, Figure 1—figure supplement 1, Figure 1—figure supplement 2 and Figure 1—figure supplement 3.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.008">http://dx.doi.org/10.7554/eLife.00288.008</a></p><span class="inline-linked-media-wrapper"><a href="[media-elife00288s001-download]"><i class="icon-download-alt"></i> Download source data<span class="inline-linked-media-filename">[figure-1—source-data-1.media-1.xlsx]</span></a></span></div></div><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.009"><div id="fig1s1" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs001-large]" class="figure-expand-popup" title="Figure 1—figure supplement 1."><img data-img="[graphic-elife00288fs001-small]" src="[graphic-elife00288fs001-medium]" alt="Figure 1—figure supplement 1."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs001-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs001-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 1—figure supplement 1.</span> <span class="caption-title">Dynamics of HSV-2 shedding over 5-min time intervals.</span><p>(<strong>A</strong>) and (<strong>B</strong>) Shedding quantity in two participants, who performed genital swabs every 5 min over 4 hr during a lesion, reveals low swab-to-swab variation in viral quantity. (<strong>C</strong>) Using data from panels (<strong>A</strong> and <strong>B</strong>), absolute mean difference in HSV DNA copies between swabs, correlated moderately with time between swabs in one participant (green line R<sup>2</sup> = 0.60) due to steady viral decay, but was more stable as a function of time in the other participant due to peaking overall viral load (blue line, R<sup>2</sup> = 0.16). (<strong>D</strong>) Using data from panels (<strong>A</strong> and <strong>B</strong>), mean difference in HSV DNA copies between swabs, was a function of time between swabs in the participants (green line R<sup>2</sup> = 0.89, and blue line R<sup>2</sup> = 0.85) presumably because viral load was generally decaying during most of the 4-hr window (green) or gradually expanding during most of the 4-hr window (blue).</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.009">http://dx.doi.org/10.7554/eLife.00288.009</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.010"><div id="fig1s2" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs002-large]" class="figure-expand-popup" title="Figure 1—figure supplement 2."><img data-img="[graphic-elife00288fs002-small]" src="[graphic-elife00288fs002-medium]" alt="Figure 1—figure supplement 2."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs002-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs002-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 1—figure supplement 2.</span> <span class="caption-title">Dynamics of HSV-2 shedding with every 2-4 hr sampling.</span><p>Shedding quantity in four participants, who performed 10 genital swabs per day over 4–5 days during a lesion reveal a characteristic saw-tooth pattern; arrows denote re-expansion. Participants had swabbing initiated upon visualization of lesions. The participant in panel d initiated swabbing ∼16 hr after lesion detection.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.010">http://dx.doi.org/10.7554/eLife.00288.010</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.011"><div id="fig1s3" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs003-large]" class="figure-expand-popup" title="Figure 1—figure supplement 3."><img data-img="[graphic-elife00288fs003-small]" src="[graphic-elife00288fs003-medium]" alt="Figure 1—figure supplement 3."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs003-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs003-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 1—figure supplement 3.</span> <span class="caption-title">Dynamics of HSV-2 shedding with every 6-hr sampling over 8 days.</span><p>Shedding quantity in episodes detected in a participant who performed four genital swabs per day over 60 days demonstrates that viral re-expansion allows for shedding prolongation.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.011">http://dx.doi.org/10.7554/eLife.00288.011</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div></div></div>

        </div>
        <div id="s2-2" class="subsection">
            <h3>Spatial features of HSV-2 replication and immunological response</h3>
            <p>Based on the observation that HSV-2 reactivation occurs simultaneously in multiple regions across the genital tract (<a class="xref-bibr" href="#bib33">Tata et al., 2010</a>), we designed a fourth cohort of patients who had daily genital swabs performed in 23 locations (<a class="xref-table" href="#tbl1">Table 1</a>, <a class="xref-boxed-text" href="#box4">Box 4</a>). This study revealed that HSV-2 is present across the entire genital tract during many episodes, although viral loads are highly variable over space and time (<a class="xref-fig" href="#fig2">Figure 2A</a>, <a class="xref-fig" href="#fig2s1">Figure 2—figure supplement 1</a>). We next demonstrated that viral re-expansion more commonly follows a period of decay during episodes with higher peak viral loads (<a class="xref-table" href="#tbl1">Table 1</a>, <a class="xref-boxed-text" href="#box5">Box 5</a>, and <a class="xref-fig" href="#fig2">Figure 2B</a>): if each peak during a prolonged shedding episodes (<a class="xref-fig" href="#fig1">Figure 1E</a>) represents viral production from a single focus of infection, then this implies that multiple spatially dispersed foci of replication (<a class="xref-fig" href="#fig2">Figure 2A</a>) may occur due to seeding from areas with high levels of HSV-2 replication. Concurrent viral expansion and clearance in spatially distinct microregions could account for saw-tooth episodes (<a class="xref-fig" href="#fig1">Figure 1E</a>), and viral migration to new regions might prolong episodes.</p>

            <div class="boxed-text" id="box4"><span class="boxed-text-label">Box 4.</span><p class="first-child">Cohort D</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.012">http://dx.doi.org/10.7554/eLife.00288.012</a></p><p>We investigated the spatial-temporal dynamics of reactivation in two patients in which we divided the genital tract into 23 separate regions and swabbed each individual for 30 consecutive days (Cohort D). During brief asymptomatic episodes, low HSV-2 quantities were confined to a few regions, in which viral load and location fluctuated over time (<a class="xref-fig" href="#fig2">Figure 2A</a>, <a class="xref-fig" href="#fig2s1">Figure 2—figure supplement 1</a>, <a class="xref-supplement-material" href="#SD2-data">Figure 2—source data 1</a>). During longer more severe episodes, HSV was detected throughout the genital tract (<a class="xref-fig" href="#fig2">Figure 2A</a>), although viral density often varied by &gt;4 logs between adjacent regions.</p></div>
            <div class="fig-inline-img-set-carousel"><div class="elife-fig-slider-wrapper"><div class="elife-fig-slider"><div class="elife-fig-slider-img elife-fig-slider-primary"><a href="#fig2"><img src="[graphic-elife00288f002-small]" alt="Figure 2."/></a></div><div class="figure-carousel-inner-wrapper"><div class="figure-carousel"><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig2s1"><img src="[graphic-elife00288fs004-small]" alt="Figure 2—figure supplement 1."/></a></div><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig2s2"><img src="[graphic-elife00288fs005-small]" alt="Figure 2—figure supplement 2."/></a></div><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig2s3"><img src="[graphic-elife00288fs006-small]" alt="Figure 2—figure supplement 3."/></a></div></div></div></div></div><div class="elife-fig-image-caption-wrapper"><div class="fig" data-doi="10.7554/eLife.00288.013"><div id="fig2" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288f002-large]" class="figure-expand-popup" title="Figure 2."><img data-img="[graphic-elife00288f002-small]" src="[graphic-elife00288f002-medium]" alt="Figure 2."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288f002-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288f002-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 2.</span> <span class="caption-title">HSV-2 replicates and is contained in widely dispersed microenvironments across the genital tract.</span><p>(<strong>A</strong>) HSV shedding quantity in a participant, who underwent daily swabs in 23 regions across the genital tract for 30 days; days without sampling are marked with an X; stars denote days with a lesion; virus is widely dispersed and several prolonged episodes with heterogeneous viral loads across the genital tract are noted. (<strong>B</strong>) Increasing probability of episode re-expansion (nonmonotonic episodes) as a function of peak episode copy number among 1020 episodes from 531 study subjects; individual peaks during episodes may represent virus from a single ulcer that can seed other regions. (<strong>C</strong>) A genital lesion consists of numerous round ulcers (black dotted circle) clustered in space; contemporaneous presence of multiple ulcers may indicate concurrent viral expansion in decay in multiple regions. (<strong>D</strong>) and (<strong>E</strong>) Immunofluorescent staining of biopsies performed (<strong>D</strong>) at the edge, and (<strong>E</strong>) 1 cm away from an ulcer 3 days post-healing; CD8+ T cells (green) at the dermal–epidermal junction (arrow) are highly localized to ulcer edge (287/mm<sup>2</sup>) and are fourfold less dense 1 cm away (72/mm<sup>2</sup>).</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.013">http://dx.doi.org/10.7554/eLife.00288.013</a></p><div class="supplementary-material" data-doi="10.7554/eLife.00288.014"><div class="supplementary-material-expansion" id="SD2-data"><span class="supplementary-material-label">Figure 2—source data 1.</span><p>Source data for Figure 2 and Figure 2—figure supplement 1.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.014">http://dx.doi.org/10.7554/eLife.00288.014</a></p><span class="inline-linked-media-wrapper"><a href="[media-elife00288s002-download]"><i class="icon-download-alt"></i> Download source data<span class="inline-linked-media-filename">[figure-2—source-data-1.media-2.xlsx]</span></a></span></div></div><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.015"><div id="fig2s1" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs004-large]" class="figure-expand-popup" title="Figure 2—figure supplement 1."><img data-img="[graphic-elife00288fs004-small]" src="[graphic-elife00288fs004-medium]" alt="Figure 2—figure supplement 1."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs004-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs004-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 2—figure supplement 1.</span> <span class="caption-title">Spatial features of HSV genital tract shedding.</span><p>HSV shedding quantity in a study participant, who underwent daily swabs in 23 regions across the genital tract for 30 days; days without sampling are marked with an X. The participant had three brief localized episodes with low viral copy number in three separate localized regions.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.015">http://dx.doi.org/10.7554/eLife.00288.015</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.016"><div id="fig2s2" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs005-large]" class="figure-expand-popup" title="Figure 2—figure supplement 2."><img data-img="[graphic-elife00288fs005-small]" src="[graphic-elife00288fs005-medium]" alt="Figure 2—figure supplement 2."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs005-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs005-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 2—figure supplement 2.</span> <span class="caption-title">Spatial features of HSV-2 lesions.</span><p>A genital lesion consists of numerous round ulcers or vesicles (black dotted circle), clustered in space.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.016">http://dx.doi.org/10.7554/eLife.00288.016</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.017"><div id="fig2s3" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs006-large]" class="figure-expand-popup" title="Figure 2—figure supplement 3."><img data-img="[graphic-elife00288fs006-small]" src="[graphic-elife00288fs006-medium]" alt="Figure 2—figure supplement 3."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs006-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs006-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 2—figure supplement 3.</span> <span class="caption-title">Spatial features of CD8+ T-cell response in genital skin.</span><p>Immunofluorescent staining of a biopsy performed (<strong>A</strong>) at the edge, and (<strong>B</strong>) 1 cm away from an ulcer 3 days post-healing. CD8+ T-cells (red) and CD4+ T-cells (green) at the dermal epidermal junction (arrow) are highly localized to ulcer edge (132/mm<sup>2</sup> and 447/mm<sup>2</sup>, respectively), and are less dense 1 cm away (91/mm<sup>2</sup> and 132/mm<sup>2</sup>, respectively).</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.017">http://dx.doi.org/10.7554/eLife.00288.017</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div></div></div>
            <div class="boxed-text" id="box5"><span class="boxed-text-label">Box 5.</span><p class="first-child">Cohort E</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.018">http://dx.doi.org/10.7554/eLife.00288.018</a></p><p>To define episode heterogeneity, we used the largest available database of HSV-2 shedding (Cohort E), which contained HSV PCR data taken from 531 HSV-2 seropositive persons who sampled the entire genital tract once daily for &gt;30 days. This dataset included 14,685 swabs and 1020 episodes of symptomatic and asymptomatic shedding (<a class="xref-bibr" href="#bib28">Schiffer et al., 2011</a>). Each episode was classified according to six features: duration, first, peak and last positive HSV genomic copy number, and initiation to peak and peak to termination slopes. The heterogeneity of each feature was captured using frequency histograms. Shedding episodes had highly variable duration (median days: 3; IQR: 1–8) and peak HSV DNA copy number (median log<sub>10</sub> copy number: 4.8; range: 2–9.2). Episode rate was 15.3/year, and per swab shedding frequency was 18% (∼3% of swabs contained 10<sup>2</sup>–10<sup>3</sup>, 10<sup>3</sup>–10<sup>4</sup>, 10<sup>4</sup>–10<sup>5</sup>, 10<sup>5</sup>–10<sup>6</sup>, 10<sup>6</sup>–10<sup>7</sup>, and &gt;10<sup>7</sup> HSV DNA copies, respectively). In Cohort E, 19% of episodes had re-expansion, which we previously defined as 0.5 log decay followed by 0.5 log re-expansion (<a class="xref-bibr" href="#bib27">Schiffer et al., 2011a</a>, <a class="xref-bibr" href="#bib28">2011b</a>).</p></div>

            <p>Further evidence for this phenomenon is that herpetic lesions consist of multiple vesicles and ulcers (<a class="xref-bibr" href="#bib4">Corey et al., 1983</a>), which occur in clusters and develop sequentially (<a class="xref-fig" href="#fig2">Figure 2C</a>, <a class="xref-fig" href="#fig2s2">Figure 2—figure supplement 2</a>), suggesting that virus from a single ulcer may lead to ancillary ulcer formation. In addition, we have previously demonstrated that host T-cell responses in the area of genital lesions are highly localized to foci of viral replication (<a class="xref-bibr" href="#bib40">Zhu et al., 2007</a>, <a class="xref-bibr" href="#bib37">2009</a>). To better classify T-cell heterogeneity over short distances, we performed biopsies in two patients from Cohort B immediately after lesion healing, and enumerated T-cell counts at the edge of an ulcer and 1 cm away. CD8+ and CD4+ T-cell densities in the genital skin were maximal at ulcer edge (<a class="xref-fig" href="#fig2">Figure 2D</a>, <a class="xref-fig" href="#fig2s3">Figure 2—figure supplement 3A</a>), and considerably lower in regions 1 cm away (<a class="xref-fig" href="#fig2">Figure 2E</a>, <a class="xref-fig" href="#fig2s3">Figure 2—figure supplement 3B</a>). Even within a 1-mm biopsy, CD8+ T cells formed cluster-like aggregates (<a class="xref-fig" href="#fig2">Figure 2D</a>, <a class="xref-fig" href="#fig2s3">Figure 2—figure supplement 3A</a>), suggesting that dynamical interactions between acquired immune cells and infected cells occur within tightly spaced microenvironments on a micrometer scale. We infer that individual ulcers, both microscopic and visible to the naked eye, may occur across a wide region of the genital tract during large outbreaks. While each ulcer and microulcer represent a site of intense interface between the virus and T cells, genital skin and mucosa between ulcers may be relatively quiescent.</p>
        </div>
        <div id="s2-3" class="subsection">
            <h3>Fitting dataset for mathematical models</h3>
            <p>To generate hypotheses that might explain these observations, we sought to design a mathematical model that precisely recreated the shedding patterns observed in our human shedding studies. Rather than fit models to complex individual episodes with erratic expansion and decay phases (<a class="xref-fig" href="#fig1">Figure 1D,E</a>), we tested competing models for their ability to reproduce key kinetic patterns from a large more generalizable patient base with genital herpes (<a class="xref-table" href="#tbl1">Table 1</a>, <a class="xref-boxed-text" href="#box5">Box 5</a>).</p>
        </div>
        <div id="s2-4" class="subsection">
            <h3>Spatial model of HSV-2 dynamics</h3>
            <p>We attempted to develop a model that could precisely capture episode heterogeneity within and between persons by recreating frequency histograms from Cohort E, and predicting key observations from Cohorts A–D, including rapid viral expansion and decay, multiple peaks per episode, and wide spatial distribution of HSV DNA. After several iterations of model development failed to reproduce key features of our empirical dataset (‘Methods’), we designed a spatial model that accounted for observed multiple concurrent foci of viral replication and immune response (<a class="xref-fig" href="#fig2">Figure 2</a>). We hypothesized that secondary ulcer formation in separate genital tract regions might occur via two mechanisms: release of HSV-2 from neuron endings may simultaneously occur in spatially distinct genital regions, or dense aggregates of cell-free particles within an ulcer may be locally infectious. Evidence for the latter hypothesis is tight correlation between episode peak copy number and probability of re-expansion in (<a class="xref-fig" href="#fig2">Figure 2B</a>). In addition, ulcers tend to be closely arrayed in space rather than widely dispersed across vulnerable regions in the genital tract. ‘Kissing lesions’, in which two ulcers form at the site of skin-to-skin contact, are a well-documented phenomenon (<a class="xref-fig" href="#fig2">Figure 2C</a>). On the other hand, ulcers do not isolate within genital tract dermatomes as occurs with zoster reactivation. These observations suggest that local seeding from epithelial lesions may be responsible for many secondary ulcers.</p>
            <p>Our spatial model included 300 regions of 6.5 mm diameter linked in a spatial arrangement of adjacent hexagons (<a class="xref-fig" href="#fig3">Figure 3A</a>, <a class="xref-fig" href="#fig3s1">Figure 3—figure supplement 1</a>). The diameter was chosen as an absolute maximal value for ulcer diameter. We populated each region with differential equations (<a class="xref-fig" href="#fig3">Figure 3B</a>, ‘Methods’) describing infected epithelial cells (I), which produced cell-associated HSV-2 (V<sub>i</sub>) at a rate <em>p</em>. Cell-associated HSV-2 was infectious to susceptible epithelial cells (S) according to an infectivity parameter β<sub>I</sub>. Infected cells died either due to direct HSV-2 killing at a rate <em>a</em>, or due to CD8+ T-cell killing at a rate (<em>f</em> × E). Cytolytic CD8+ T cell (E) expanded at a maximal rate θ. CD8+ expansion rate increased according to number of infected cells, and was half-maximal (θ/2) at a threshold value of infected cells, <em>r.</em> Cell-associated HSV-2 converted to cell-free HSV-2 (V<sub>e</sub>) following cell lysis. Cell-free viruses and CD8+ T cells decayed at fixed rates (<em>c</em> and δ) within each region. We assumed that viruses (V<sub>neu</sub>) were randomly released into 300 regions by neurons at a rate ϕ, predicted by a previous model (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>), and that these viruses could initiate an ulcer in each reason by infecting an epithelial cell.</p>

            <div class="fig-inline-img-set-carousel"><div class="elife-fig-slider-wrapper"><div class="elife-fig-slider"><div class="elife-fig-slider-img elife-fig-slider-primary"><a href="#fig3"><img src="[graphic-elife00288f003-small]" alt="Figure 3."/></a></div><div class="figure-carousel-inner-wrapper"><div class="figure-carousel"><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig3s1"><img src="[graphic-elife00288fs007-small]" alt="Figure 3—figure supplement 1."/></a></div></div></div></div></div><div class="elife-fig-image-caption-wrapper"><div class="fig" data-doi="10.7554/eLife.00288.019"><div id="fig3" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288f003-large]" class="figure-expand-popup" title="Figure 3."><img data-img="[graphic-elife00288f003-small]" src="[graphic-elife00288f003-medium]" alt="Figure 3."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288f003-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288f003-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 3.</span> <span class="caption-title">Mathematical model.</span><p>(<strong>A</strong>) Microregions are linked virally because cell-free HSV-2 can seed surrounding regions, and immunologically based on overlapping CD8+ T-cell densities between regions (not shown). (<strong>B</strong>) Schematic for HSV-2 infection within a single genital tract microenvironment. Equations capture seeding of epithelial cells by neuronal HSV-2, replication of HSV-2 within epithelial cells, viral spread to other epithelial cells, cytolytic CD8+ T-cell response to infected cells, transition of cell-associated HSV-2 to cell-free HSV-2 following lysis of infected cells, and elimination of free virus and infected cells.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.019">http://dx.doi.org/10.7554/eLife.00288.019</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.020"><div id="fig3s1" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs007-large]" class="figure-expand-popup" title="Figure 3—figure supplement 1."><img data-img="[graphic-elife00288fs007-small]" src="[graphic-elife00288fs007-medium]" alt="Figure 3—figure supplement 1."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs007-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs007-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 3—figure supplement 1.</span> <span class="caption-title">Spatial mathematical model.</span><p>Viruses produced from neurons (green), cell-associated viruses from epidermal cells (yellow), and cell-free viruses (orange) that form after rupture of epidermal cells, are distinguished in the model. Neuron-derived viruses are released throughout the genital tract and are responsible for ulcer initiation within specific regions (grey hexagons). Cell-associated HSV particles contribute to ulcer expansion (white circle) within a region. Cell-free particles initiate secondary ulcers in adjacent regions (upper right) leading to concurrent ulcers where HSV production occurs. Cytolytic CD8+ T-cell (purple circles) response is localized within each region. Regions have a maximum diameter of 6.5 mm. However, distance between regions is considered in terms of immunologic co-dependence rather than a physical distance. Seven of 300 total model regions are illustrated.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.020">http://dx.doi.org/10.7554/eLife.00288.020</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div></div></div>

            <p>Adjacent regions in the model were linked virally. Cell-associated HSV (V<sub>i</sub>) drove spread within an ulcer in a single region, while cell-free HSV (V<sub>e</sub>) could initiate new ulcers at infectivity β<sub>e</sub>, but only in six contiguous regions surrounding a productive ulcer (<a class="xref-fig" href="#fig3">Figure 3A</a>, <a class="xref-fig" href="#fig3s1">Figure 3—figure supplement 1</a>). Based on our observation in cell culture that in a single cell infected by a single virus, viral replication does not occur until approximately 12–16 hr, a fixed time delay parameter (ε) was included for ulcer formation.</p>
            <p>The physical distance between regions was not explicitly considered because the 300 regions were not intended to capture the complex three-dimensional topography of genital skin. Rather, the distance between regions was captured in immunologic terms. Based on the gradient of CD8+ T-cell density as distance increases from an ulcer edge (<a class="xref-fig" href="#fig2">Figure 2D,E</a>), we assumed that contiguous regions might be immunologically codependent, by including a new fitting parameter (ρ) to estimate the extent that CD8+ T-cell density in contiguous regions affected CD8+ T-cell density within a new ulcer region (‘Methods’). Contiguous regions in the model were therefore assumed to be far enough away for new ulcers to initiate but potentially close enough to be effected by neighboring immune responses.</p>
        </div>
        <div id="s2-5" class="subsection">
            <h3>Model fitting</h3>
            <p>We solved our model by fitting to the data and assuming either 5 or 10 above parameter values as unknown (‘Methods’). In both cases, model output closely reproduced the data within Cohort E, including quantitative shedding frequency (<a class="xref-fig" href="#fig4">Figure 4A</a>), as well as episode rate (<a class="xref-fig" href="#fig4">Figure 4B</a>), median initiation to peak and peak to termination slopes (<a class="xref-fig" href="#fig4">Figure 4C</a>), durations (<a class="xref-fig" href="#fig4">Figure 4D</a>), and first (<a class="xref-fig" href="#fig4">Figure 4E</a>), last (<a class="xref-fig" href="#fig4">Figure 4F</a>), and peak HSV DNA copy numbers (<a class="xref-fig" href="#fig4">Figure 4G</a>, <a class="xref-supplement-material" href="#SD3-data">Figure 4—source data 1</a>). We also performed a sensitivity analysis using 500 episode (∼30 years) simulations in which single parameter values were adjusted to arrive at narrow ranges for parameter values that reproduced our data (<a class="xref-table" href="#tbl2">Table 2</a>). These parameter values were generally within an order of magnitude of previous parameter estimates (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>).</p>

            <div class="fig-inline-img-set-carousel"><div class="elife-fig-slider-wrapper"><div class="elife-fig-slider"><div class="elife-fig-slider-img elife-fig-slider-primary"><a href="#fig4"><img src="[graphic-elife00288f004-small]" alt="Figure 4."/></a></div><div class="figure-carousel-inner-wrapper"><div class="figure-carousel"><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig4s1"><img src="[graphic-elife00288fs008-small]" alt="Figure 4—figure supplement 1."/></a></div></div></div></div></div><div class="elife-fig-image-caption-wrapper"><div class="fig" data-doi="10.7554/eLife.00288.021"><div id="fig4" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288f004-large]" class="figure-expand-popup" title="Figure 4."><img data-img="[graphic-elife00288f004-small]" src="[graphic-elife00288f004-medium]" alt="Figure 4."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288f004-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288f004-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 4.</span> <span class="caption-title">The spatial model reproduces all shedding episode characteristics.</span><p>Colored bars represent results from (<strong>A</strong>) 14,685 genital swabs and (<strong>B</strong>–<strong>G</strong>) 1020 shedding episodes from 531 study participants. The model simulation, represented with black bars in each panel, continued until 1020 episodes were generated; model sampling occurred every 24 hr as in the clinical protocol. Model output reproduced (<strong>A</strong>) quantitative shedding frequency as well as (<strong>B</strong>) rate, (<strong>C</strong>) median initiation to peak and peak to termination slopes, (<strong>D</strong>) Duration, (<strong>E</strong>) first HSV DNA copy number, (<strong>F</strong>) last HSV DNA copy number, and (<strong>G</strong>) peak HSV DNA copy number of episodes.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.021">http://dx.doi.org/10.7554/eLife.00288.021</a></p><div class="supplementary-material" data-doi="10.7554/eLife.00288.022"><div class="supplementary-material-expansion" id="SD3-data"><span class="supplementary-material-label">Figure 4—source data 1.</span><p>Source data for Figure 4.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.022">http://dx.doi.org/10.7554/eLife.00288.022</a></p><span class="inline-linked-media-wrapper"><a href="[media-elife00288s003-download]"><i class="icon-download-alt"></i> Download source data<span class="inline-linked-media-filename">[figure-4—source-data-1.media-3.xlsx]</span></a></span></div></div><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.023"><div id="fig4s1" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs008-large]" class="figure-expand-popup" title="Figure 4—figure supplement 1."><img data-img="[graphic-elife00288fs008-small]" src="[graphic-elife00288fs008-medium]" alt="Figure 4—figure supplement 1."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs008-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs008-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 4—figure supplement 1.</span> <span class="caption-title">Continuous sampling of spatial model output reveals more accurate measures of episode characteristics.</span><p>We subjected a 30-year simulation to daily and continuous sampling. (<strong>A</strong>) Median initiation to peak slope, and (<strong>B</strong>) peak to termination slopes increased substantially with continual sampling. (<strong>C</strong>) Shedding frequency was similar regardless of sampling frequency. (<strong>D</strong>) Continuous sampling detected 842 episodes (28.1/year) vs 450 episodes (15.0/year) with daily sampling. The 392 additional episodes were all less than a day in duration and mostly &lt;10<sup>4</sup> peak HSV DNA copies per milliliter, skewing the distributions of (<strong>E</strong>) episode duration and (<strong>F</strong>) peak HSV DNA copy number. (<strong>G</strong>) Total number of episodes at low and high-peak copy numbers increased with continual sampling.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.023">http://dx.doi.org/10.7554/eLife.00288.023</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div></div></div>
            <div class="table-expansion" id="tbl2"><div class="table-caption"><span class="table-label">Table 2.</span> <p>Parameter ranges that result in accurate reproduction of model outcomes</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.024">http://dx.doi.org/10.7554/eLife.00288.024</a></p><div class="sb-div caption-clear"></div></div><table><thead><tr><td rowspan="2">Parameter</td><td rowspan="2">Units</td><td rowspan="2">Symbol</td><td rowspan="2">Best fit value</td><td colspan="2">Good fit</td><td colspan="2">Average fit</td></tr><tr><td>Lower limit</td><td>Upper limit</td><td>Lower limit</td><td>Upper limit</td></tr></thead><tbody><tr><td>Cell-associated HSV infectivity</td><td>DNA copy days/cell (viruses needed per day to infect one adjacent cell)</td><td>β<sub>i</sub></td><td>5.4e<sup>−8</sup> (111)</td><td>4.86e<sup>−8</sup> (123)</td><td>7.83e<sup>−8</sup> (76)</td><td>3.78e<sup>−8</sup> (158)</td><td>1.32e<sup>−7</sup> (45)</td></tr><tr><td>Cell-free HSV infectivity</td><td>DNA copy days/cell (viruses needed per day to initiate one new ulcer)</td><td>β<sub>e</sub></td><td>2.65e<sup>−11</sup> (2.26e<sup>5</sup>)</td><td>1.73e<sup>−11</sup> (3.46e<sup>5</sup>)</td><td>2.78e<sup>−11</sup> (2.15e<sup>5</sup>)</td><td>3.98e<sup>−12</sup> (1.50e<sup>6</sup>)</td><td>5.04e<sup>−11</sup> (1.19e<sup>5</sup>)</td></tr><tr><td>Epidermal HSV replication rate</td><td>HSV DNA copies per cell per day</td><td><em>p</em></td><td>1.03e<sup>5</sup></td><td>7.21e<sup>4</sup></td><td>1.7e<sup>5</sup></td><td>5.15e<sup>4</sup></td><td>1.96e<sup>5</sup></td></tr><tr><td>Neuronal release rate</td><td>HSV DNA copies per day per genital tract</td><td>ϕ</td><td>82</td><td>45</td><td>90</td><td>41</td><td>123</td></tr><tr><td>Free-viral decay rate</td><td>Per day (half-life, hours)</td><td><em>c</em></td><td>8.8 (1.9)</td><td>7.0 (2.4)</td><td>9.7 (1.7)</td><td>6.2 (2.7)</td><td>12.3 (1.4)</td></tr><tr><td>Maximal CD8+ T-cell expansion rate</td><td>Per day</td><td>θ</td><td>2.84</td><td>1.85</td><td>3.27</td><td>1.85</td><td>5.25</td></tr><tr><td>CD8+ T-cell decay rate</td><td>Per day (half-life, days)</td><td>δ</td><td>1.47e<sup>−3</sup> (471)</td><td>1.12e<sup>−3</sup> (619)</td><td>1.69e<sup>−3</sup> (409)</td><td>6.64e<sup>−4</sup> (1040)</td><td>2.21e<sup>−3</sup> (314)</td></tr><tr><td>CD8+ T-cell local recognition</td><td>Infected cells at which θ is half maximal</td><td><em>r</em></td><td>42</td><td>30</td><td>47</td><td>4</td><td>74</td></tr><tr><td>CD8+ regional codependence</td><td>0 = no codependence, 1 =full codependence</td><td>ρ</td><td>0.69</td><td>0.59</td><td>0.86</td><td>0.38</td><td>0.86</td></tr><tr><td>Viral production lag</td><td>Days</td><td>ε</td><td>0.96</td><td>0.53</td><td>1.1</td><td>0.34</td><td>1.1</td></tr></tbody></table></div>

        </div>
        <div id="s2-6" class="subsection">
            <h3>Frequent heterogeneous episodes with extremely rapid expansion, rapid decay, and multiple peaks</h3>
            <p>The model was next evaluated for its ability to predict other key features of genital shedding identified in Cohorts A–D. To explore the dynamics of simulated episodes with higher granularity, we varied sampling frequency of model output to include continuous sampling. The model predicted the empirically derived finding that daily sampling substantially underestimates expansion (<a class="xref-fig" href="#fig4s1">Figure 4—figure supplement 1A</a>) and clearance slopes (<a class="xref-fig" href="#fig4s1">Figure 4—figure supplement 1B</a>). Among 842 simulated episodes with sampling every 0.001 days, median initiation to peak expansion rate was 25.5 log<sub>10</sub> HSV DNA copies per day (versus 20.3 log<sub>10</sub> copies per day with 6-hr sampling in Cohort C), implying that during the early expansion phase, HSV DNA levels increased 10-fold every 57 min and doubled every 17 min. There was also rapid late clearance (−7.4 log<sub>10</sub> copies per day vs −8.7 log<sub>10</sub> copies per day in Cohort C).</p>
            <p>We simulated the model until 100 episodes with &gt;10,000 total infected cells within one ulcer were generated. Number of infected cells and viral load peaked at a median of 13.5 hr, predicting the finding from Cohort C that transition from expansion to clearance occurs ∼12 hr after episode commencement. To explain episodes of &gt;4 days, the model predicted multiple peaks within single episodes. In addition, prolonged simulated episodes had wide concurrent dispersal of virus (<a class="xref-other" href="#video1">Video 1</a>).</p>

            <div class="media video-content" id="video1"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video1-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video1-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 1.</span> <span class="caption-title">Spatiotemporal demonstration of a 10-day episode.</span><p class="first-child">The left panel represents total cell-free HSV DNA copies per milliliter present over time. The right panel represents spatial spread of virus during the episode, each hexagon contains one region of shedding and virus spreads to contiguous regions. Amount of virus within a single region is displayed according to a heat map adjacent to the spatial map.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.025">http://dx.doi.org/10.7554/eLife.00288.025</a></p></div></div>

            <p>While daily sampling accurately measured quantitative shedding frequency with continuous sampling (<a class="xref-fig" href="#fig4s1">Figure 4—figure supplement 1C</a>), the model predicted that more frequent sampling increased episode detection by ∼50% (<a class="xref-fig" href="#fig4s1">Figure 4—figure supplement 1D</a>) and proportion of episodes with low duration (<a class="xref-fig" href="#fig4s1">Figure 4—figure supplement 1E</a>) and low peak copy number (<a class="xref-fig" href="#fig4s1">Figure 4—figure supplement 1F</a>). The model also predicted the finding from Cohort C that ∼75% of HSV-2 episodes are asymptomatic, brief (&lt;48 hr), and associated with a low peak copy number (<a class="xref-fig" href="#fig4s1">Figure 4—figure supplement 1F</a>) (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>). There were higher absolute numbers of episodes with high copy number (&gt;10<sup>7</sup> HSV DNA copies) because daily sampling rarely captured true episode peak (<a class="xref-fig" href="#fig4s1">Figure 4—figure supplement 1G</a>).</p>
        </div>
        <div id="s2-7" class="subsection">
            <h3>Efficient cell-to-cell HSV-2 spread within a single epidermal ulcer</h3>
            <p>Our estimates of parameter values suggest that individual cell-associated viruses are rapidly produced by epidermal cells and are highly infectious to surrounding cells. Epidermal cells produced 72,000–170,000 HSV DNA copies per day; one of 76–123 cell-associated DNA particles infected a neighboring cell per day at ulcer onset (<a class="xref-table" href="#tbl2">Table 2</a>). When we simulated the model until 100 episodes with &gt;10,000 total infected cells within a single ulcer were generated, median times to 100, 1000, and 10,000 total infected cells were 5.6, 8.7, and 12.9 hr, respectively. Median time interval from becoming infected to infecting a new cell during the expansion phase of the 100 simulated episodes was 6.8 hr; median time from becoming infected to infecting a first surrounding cell was 59 min. This finding was surprising because in culture, HSV replication occurs within newly infected cells 4–6 hr post-infection (<a class="xref-bibr" href="#bib23">Roizman, 2007</a>). Our data suggest that in vivo an HSV-infected keratinocyte might become locally infectious by spreading the virus even before replication initiates within this cell, a phenomenon recently described for vaccinia (<a class="xref-bibr" href="#bib8">Doceul et al., 2010</a>).</p>
        </div>
        <div id="s2-8" class="subsection">
            <h3>Episode prolongation due to secondary ulcer formation</h3>
            <p>The model results suggested that secondary ulcer formation by cell-free particles is crucial for episode prolongation. In 500 simulated episodes, duration was a function of total ulcers (<a class="xref-fig" href="#fig5">Figure 5A</a>). We analyzed a simulated prolonged episode (10 days) in detail to assess kinetics within single ulcers: free viral production from the initial viral ulcer lasted 3 days (<a class="xref-fig" href="#fig5">Figure 5B</a>); infected cells were eliminated within 24 hr (<a class="xref-fig" href="#fig5">Figure 5C,D</a>); peak viral load and number of infected cells were 8.1 log<sub>10</sub> HSV DNA copies and 4.1 log<sub>10</sub> cells, respectively, although total amount of viral DNA and infected cells produced within the ulcer were considerably higher (8.5 log<sub>10</sub> HSV DNA copies and 4.9 log<sub>10</sub> cells, respectively), reflecting rapid viral and infected cell turnover. Virions generated during the primary ulcer promoted formation of additional ulcers (<a class="xref-fig" href="#fig5">Figure 5E</a> and <a class="xref-other" href="#video1">Videos 1, 2</a>). Before episode termination, 24 total ulcers were produced. The median duration and peak viral production of the 24 ulcers were 2.5 days (range: 0.6–4.6 days) and 5.2 log<sub>10</sub> HSV DNA copies (range: 2.0–8.0 log<sub>10</sub>), respectively. We calculated diameter of each ulcer during simulations based on the number of missing cells due to death from infection, and defined a lesion as any episode with a &gt;1 mm ulcer (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>). Consistent with clinical observations in Cohort D, simulated ulcer size did not exceed 5 mm (<a class="xref-bibr" href="#bib4">Corey et al., 1983</a>), and many failed to reach the clinical threshold of 1 mm in diameter (<a class="xref-other" href="#video3">Video 3</a>).</p>

            <div id="fig5" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288f005-large]" class="figure-expand-popup" title="Figure 5."><img data-img="[graphic-elife00288f005-small]" src="[graphic-elife00288f005-medium]" alt="Figure 5."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288f005-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288f005-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 5.</span> <span class="caption-title">Containment of infected cells within a single ulcer is extremely rapid, although secondary ulcers explain prolonged episodes.</span><p>(<strong>A</strong>) Episode duration was a function of the number of ulcers before episode termination during 500 simulated episodes. (<strong>B</strong>)–(<strong>E</strong>) A 10-day simulated episode consisting of 24 ulcers: (<strong>B</strong>) Total cell-free virus (red) over time reflects the saw-tooth pattern of prolonged episodes; virus produced from the initial ulcer (red dotted line) was eliminated within 3 days. (<strong>C</strong>) Infected cells were eliminated from the initial viral ulcer (green dotted line) within 1 day and there were four periods during the episode when no infected cells were present. (<strong>D</strong>) Cell-free virus (red), cell-associated virus (blue), and infected cells (green) were eliminated from the primary ulcer with different kinetics; infected cells peaked at 13 hr and were extinguished in &lt;24 hr (<strong>E</strong>) Secondary ulcers prolonged episodes; each thin line represents HSV-2 production from a specific region.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.026">http://dx.doi.org/10.7554/eLife.00288.026</a></p><div class="sb-div caption-clear"></div></div></div></div></div>
            <div class="media video-content" id="video2"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video2-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video2-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 2.</span> <span class="caption-title">Spatiotemporal demonstration of a 14-day episode according to viral production within each single region.</span><p class="first-child">The left panel represents cell-free HSV DNA measured over time with each region's production demonstrated with a different color. The right panel represents spatial spread of virus during the episode; colors in the right panel correspond to those in the left panel. Amount of virus within a region is displayed according to a heat map adjacent to the spatial map.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.027">http://dx.doi.org/10.7554/eLife.00288.027</a></p></div></div>
            <div class="media video-content" id="video3"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video3-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video3-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 3.</span> <span class="caption-title">Spatiotemporal demonstration of infected cell and viral spread during a 6-day episode.</span><p class="first-child">The upper left panel represents spatial spread of cell-free virus during the episode; the upper right panel represents spatial spread of cell-associated virus during the episode; the bottom left panel represents spatial spread of infected cells during the episode; the bottom right panel represents ulcer formation during the episode, ulcers turn from black to red when diameter exceeds 1 mm; quantities are displayed according to a heat map adjacent to each spatial map. There is more rapid decay of infected cells and cell-associated particles than of cell-free particles within each region. Visible ulcers persist after viral production terminates within a region.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.028">http://dx.doi.org/10.7554/eLife.00288.028</a></p></div></div>

            <p>Parameter estimates suggest that cell-free HSV was ∼2000-fold less infectious than cell-associated HSV (<a class="xref-table" href="#tbl2">Table 2</a>), possibly because HSV in the aqueous environment outside of the cell must contend with genital secretion flow kinetics, mucous, low pH, and neutralizing or ADCC-like antibodies, while cell-associated HSV avoids these hazards by congregating within cellular tight junctions (<a class="xref-bibr" href="#bib3">Collins and Johnson, 2003</a>). In nonmucosal regions, a layer of dead cells that do not support viral replication protects underlying nucleated keratinocytes in the epidermis against infection, further decreasing the probability that cell-free HSV will contact a viral entry receptor.</p>
            <p>Due to slower elimination of cell-free particles, the model predicted that the number of ulcers containing cell-free particles often exceeded that of ulcers populated by HSV-infected cells, thus prolonging the opportunity for secondary seeding (<a class="xref-other" href="#video3">Video 3</a>). To address this question, we conducted a 365-day simulation with infectious and noninfectious cell-free HSV DNA (<a class="xref-other" href="#video4">Videos 4 and 5</a>) and found that noninfectious cell-free HSV, as may occur in the presence of comprehensive neutralizing antibody protection, resulted in a decrease in episode duration and shedding frequency from 18.2% to 10.8%.</p>

            <div class="media video-content" id="video4"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video4-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video4-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 4.</span> <span class="caption-title">Spatiotemporal demonstration of 365 days of simulated shedding.</span><p class="first-child">The left panel represents total cell-free HSV DNA copies per milliliter present over time. The right panel represents spatial spread of virus during the episode; each hexagon contains one region of shedding and virus spreads to contiguous regions. Amount of virus within a single region is displayed according to a heat map adjacent to the spatial map. The simulation is notable for episodes of variable duration and peak HSV DNA copy number. Prolonged episodes at days 8, 38, 136, 158, and 288, display several re-expansion phases.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.029">http://dx.doi.org/10.7554/eLife.00288.029</a></p></div></div>
            <div class="media video-content" id="video5"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video5-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video5-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 5.</span> <span class="caption-title">Spatiotemporal demonstration of 365 days of simulated shedding with noninfectious cell-free particles.</span><p class="first-child">The left panel represents total cell-free HSV DNA copies per milliliter present over time. The right panel represents spatial spread of virus during the episode; each hexagon contains one region of shedding. Amount of virus within a single region is displayed according to a heat map adjacent to the spatial map. The simulation is notable for lack of prolonged episodes and lack of episode re-expansion.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.030">http://dx.doi.org/10.7554/eLife.00288.030</a></p></div></div>

        </div>
        <div id="s2-9" class="subsection">
            <h3>Localized CD8+ T-cell density as a key determinant of episode severity</h3>
            <p>In spatial simulations, despite rapid initial accumulation, infected cells were eliminated rapidly, highlighting the intense local immune response (<a class="xref-fig" href="#fig5">Figure 5C,D</a>). In the 100 simulated episodes described above, the median time to death of 10,000, 1000, or 100 infected cells was 13.7, 9.7, or 6.9 hr, respectively. We previously described an inverse relationship between HSV-2 episode severity and CD8+ T-cell density at the site of reactivation (<a class="xref-bibr" href="#bib26">Schiffer et al., 2010</a>). Our models predict that CD8+ T-cell density is an inverse correlate of the reproductive number (the number of cells infected by the initially infected cell) in a region. Once T-cell density reaches a certain threshold, R falls below 1 and HSV-containing cells are cleared before infection of 10 cells can occur (<a class="xref-bibr" href="#bib26">Schiffer et al., 2010</a>).</p>
            <p>We performed a global sensitivity analysis of 500 simulated episodes (‘Methods’) to evaluate drivers of single episode severity. CD8+ T-cell density within the initially infected region was the most predictive parameter of episode peak viral load and duration. Increased viral replication rate in epidermal cells and decreased CD8+ T-cell expansion rate also correlated with higher peak viral production, while increased viral infectivity and replication rate correlated with longer episode duration (<a class="xref-table" href="#tbl3">Table 3</a>).</p>

            <div class="table-expansion" id="tbl3"><div class="table-caption"><span class="table-label">Table 3.</span> <p>Predictive model parameters for key model outcomes</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.031">http://dx.doi.org/10.7554/eLife.00288.031</a></p><div class="sb-div caption-clear"></div></div><table><thead><tr><td rowspan="2"></td><td colspan="2">Single episode features</td><td colspan="2">Long-term shedding features</td></tr><tr><td>Peak viral load</td><td>Duration</td><td>Shedding rate</td><td>Episode rate</td></tr></thead><tbody><tr><td>CD8+ T-cell density at reactivation site</td><td>−0.56</td><td>−0.47</td><td>NA</td><td>NA</td></tr><tr><td>Cell-associated HSV infectivity</td><td>—</td><td>0.12</td><td>—</td><td>0.13</td></tr><tr><td>Cell-free HSV infectivity</td><td>—</td><td>—</td><td>—</td><td>—</td></tr><tr><td>Epidermal cell replicate rate</td><td>0.13</td><td>0.14</td><td>−0.25</td><td>−0.31</td></tr><tr><td>Neuronal release rate</td><td>—</td><td>—</td><td>0.43</td><td>0.55</td></tr><tr><td>Free-viral decay rate</td><td>—</td><td>—</td><td>−0.2</td><td>—</td></tr><tr><td>Maximal CD8+ T-cell expansion rate</td><td>−0.09</td><td>—</td><td>0.37</td><td>0.51</td></tr><tr><td>CD8+ T-cell decay rate</td><td>—</td><td>0.09</td><td>—</td><td>−0.16</td></tr><tr><td>CD8+ T-cell local recognition</td><td>—</td><td>—</td><td>—</td><td>—</td></tr><tr><td>CD8+ regional co-dependence</td><td>—</td><td>—</td><td>0.32</td><td>0.34</td></tr><tr><td>Viral production lag</td><td>—</td><td>—</td><td>0.24</td><td>0.23</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn"><p>Partial correlation coefficients are listed only for parameters that are found to improve predictive effect on outcomes using Akaike information criteria models. Episode features are from 500 single episode simulations. Long-term shedding outcomes were measured over 10-years during 500 simulations.</p></li></ul></div></div>

            <p>These findings suggest that local T-cell density at the site of neuronal HSV-2 release into genital skin determines short-term viral trajectories. When high CD8+ T-cell density regions enclosed the initial infected region, spread was limited (<a class="xref-other" href="#video6">Video 6</a>), whereas more prolonged episodes followed a serpiginous route along low CD8+ T-cell density pathways before encountering a high-density dead end (<a class="xref-other" href="#video7">Video 7</a>). Episodes of &gt;10 days occurred when large genital tract areas were at low CD8+ T-cell density (R &gt; 1) (<a class="xref-other" href="#video8">Video 8</a>). HSV replication sometimes terminated at edge regions, which is biologically realistic because herpetic lesions rarely form in skin surrounding the genital tract.</p>

            <div class="media video-content" id="video6"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video6-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video6-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 6.</span> <span class="caption-title">Spatiotemporal demonstration of immune response to a pair of 2-day episodes.</span><p class="first-child">The upper left panel represents total cell-free HSV DNA copies per milliliter present over time. The upper right panel represents spatial spread of cell-free virus during the episode. The lower left panel represents CD8+ T-cell density within each region. The lower right panel indicates reproductive number within each region. Quantities are displayed according to a heat map adjacent to each spatial map. Areas with high CD8+ T-cell levels and low reproductive numbers do not support high-level viral production. The simulated episodes are short because virus does not spread from the initial plaque to adjacent regions with high CD8+ T-cell density and reproductive numbers less than one.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.032">http://dx.doi.org/10.7554/eLife.00288.032</a></p></div></div>
            <div class="media video-content" id="video7"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video7-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video7-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 7.</span> <span class="caption-title">Spatiotemporal demonstration of immune response to a 7-day episode.</span><p class="first-child">The upper left panel represents total cell-free HSV DNA copies per milliliter present over time. The upper right panel represents spatial spread of cell-free virus during the episode. The lower left panel represents CD8+ T-cell density within each region. The lower right panel indicates reproductive number within each region. Quantities are displayed according to a heat map adjacent to each spatial map. The simulated episode is medium length because virus spreads from the initial region to adjacent regions with low CD8+ T-cell density and reproductive numbers less than one, but is ultimately contained when it reaches an anatomic edge region outside of the genital tract, and regions of high CD8+ T-cell density and low reproductive number within the genital tract.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.033">http://dx.doi.org/10.7554/eLife.00288.033</a></p></div></div>
            <div class="media video-content" id="video8"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video8-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video8-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 8.</span> <span class="caption-title">Spatiotemporal demonstration of immune response to a 14-day episode.</span><p class="first-child">The upper left panel represents total cell-free HSV DNA copies per milliliter present over time. The upper right panel represents spatial spread of cell-free virus during the episode. The lower left panel represents CD8+ T-cell density within each region. The lower right panel indicates reproductive number within each region. Quantities are displayed according to a heat map adjacent to each spatial map. The simulated episode is prolonged because virus spreads from the initial region to adjacent regions with low CD8+ T-cell density and reproductive numbers less than one, and is not contained until many regions of the genital tract are infected.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.034">http://dx.doi.org/10.7554/eLife.00288.034</a></p></div></div>

            <p>After prolonged episodes, portions of the genital tract were effectively immunized against high-copy episodes for a short period, while other regions remained susceptible to substantial spread of virus. Over years, CD8+ T-cell and reproductive number spatial patterns cycled intermittently (<a class="xref-other" href="#video9">Video 9</a>), suggesting that HSV causes a chronic dynamic inflammatory state in the genital tract. These predictions are consistent with empirical studies of biopsied tissue of HSV-2–infected patients (<a class="xref-bibr" href="#bib6">Cunningham et al., 1985</a>; <a class="xref-bibr" href="#bib40">Zhu et al., 2007</a>, <a class="xref-bibr" href="#bib41">2009</a>), and may explain relatively stable genital shedding rates seen within infected persons over decades.</p>

            <div class="media video-content" id="video9"><div class="media-inline video-inline"><div class="elife-inline-video">[video-video9-inline]<div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="[video-video9-download]">Download Video</a></span></div></div></div><div class="media-caption"><span class="media-label">Video 9.</span> <span class="caption-title">Spatiotemporal demonstration of immune response over 20 years.</span><p class="first-child">The left panel represents CD8+ T-cell density within each region. The right panel indicates reproductive number within each region. Quantities are displayed according to a heat map adjacent to each spatial map. CD8+ T-cells expand rapidly at sites of episodes and then decay slowly over time, correlating with decreases and increases in reproductive number respectively. CD8+ T-cell and reproductive number spatial patterns cycle intermittently between a patchwork of heterogeneous density, broad low density, broad high density, and stark division between regions of high and low density. However, CD8+ T-cell density is virtually always high in at least some regions of the genital tract.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.035">http://dx.doi.org/10.7554/eLife.00288.035</a></p></div></div>

        </div>
        <div id="s2-10" class="subsection">
            <h3>Widely dispersed spatial drip of HSV-2 from neurons into genital tissue</h3>
            <p>To better define viral reactivation and latency patterns that best explain our shedding datasets, we altered the frequency and spatial dispersion of neuronal HSV release in the model; genital shedding trends were unchanged if HSV was released continuously or in daily, every 3-day, or weekly pulses, provided that the average HSV DNA release rate remained ∼82/day, and that release occurred randomly throughout the genital tract (<a class="xref-fig" href="#fig6">Figure 6</a>). Simulations in which release occurred continuously at 82 HSV DNA particles per day into 1, 5, or 20 randomly selected regions resulted in a poor fit, due to a higher percentage of rapidly contained episodes and lower shedding rate (<a class="xref-fig" href="#fig6s1">Figure 6—figure supplement 1</a>). HSV-2 release into 50 geographically clustered regions within a confined area led to lower episode rate, less shedding, and poorer overall fit (<a class="xref-fig" href="#fig6s2">Figure 6—figure supplement 2</a>). These data suggest that neuronal HSV release occurs at a frequent slow trickle across wide regions of the genital tract.</p>

            <div class="fig-inline-img-set-carousel"><div class="elife-fig-slider-wrapper"><div class="elife-fig-slider"><div class="elife-fig-slider-img elife-fig-slider-primary"><a href="#fig6"><img src="[graphic-elife00288f006-small]" alt="Figure 6."/></a></div><div class="figure-carousel-inner-wrapper"><div class="figure-carousel"><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig6s1"><img src="[graphic-elife00288fs009-small]" alt="Figure 6—figure supplement 1."/></a></div><div class="elife-fig-slider-img elife-fig-slider-secondary"><a href="#fig6s2"><img src="[graphic-elife00288fs010-small]" alt="Figure 6—figure supplement 2."/></a></div></div></div></div></div><div class="elife-fig-image-caption-wrapper"><div class="fig" data-doi="10.7554/eLife.00288.036"><div id="fig6" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288f006-large]" class="figure-expand-popup" title="Figure 6."><img data-img="[graphic-elife00288f006-small]" src="[graphic-elife00288f006-medium]" alt="Figure 6."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288f006-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288f006-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 6.</span> <span class="caption-title">Random spatial dispersion of viral particles from neurons reproduced the full diversity of episode characteristics if particles were released continuously, daily, or weekly from neurons.</span><p>White circles represent results from (<strong>A</strong>) 14,685 genital swabs and (<strong>B</strong>–<strong>G</strong>) 1020 shedding episodes from 531 study participants (<a class="xref-fig" href="#fig4">Figure 4</a>). The model simulations represented with colored bars in each panel continued until 1020 episodes were generated. Sampling occurred every 24 hr as in the clinical protocols. Model output with release of virus randomly throughout the 300 regions on a continuous (pink), daily (purple), every 3 days (blue), and weekly (orange) basis at an average rate of 82 HSV DNA particles per day reproduced (<strong>A</strong>) quantitative shedding frequency and episode, (<strong>B</strong>) rate, (<strong>C</strong>) median initiation to peak slope and peak to termination slopes, (<strong>D</strong>) Duration, (<strong>E</strong>) first HSV DNA copy number, (<strong>F</strong>) last HSV DNA copy number, and (<strong>G</strong>) peak HSV DNA copy number.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.036">http://dx.doi.org/10.7554/eLife.00288.036</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.037"><div id="fig6s1" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs009-large]" class="figure-expand-popup" title="Figure 6—figure supplement 1."><img data-img="[graphic-elife00288fs009-small]" src="[graphic-elife00288fs009-medium]" alt="Figure 6—figure supplement 1."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs009-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs009-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 6—figure supplement 1.</span> <span class="caption-title">Random spatial dispersion of viral particles from neurons reproduced the full diversity of episode characteristics during simulations in which particles were released into only a minority of the 300 modeled regions.</span><p>White circles represent results from (<strong>A</strong>) 14,685 genital swabs and (<strong>B</strong>–<strong>G</strong>) 1020 shedding episodes from 531 study participants (Figure 4). The model simulations represented with colored bars in each panel continued until 1020 episodes were generated. Sampling occurred every 24 hr as in the clinical protocols. Model output with continuous release (82 HSV DNA particles per day) of virus randomly to 300 (pink), 100 (blue), and 50 (brown) regions reproduced (<strong>A</strong>) quantitative shedding frequency and episode, (<strong>B</strong>) rate, (<strong>C</strong>) median initiation to peak slope and peak to termination slopes, (<strong>D</strong>) Duration, (<strong>E</strong>) first HSV DNA copy number, (<strong>F</strong>) last HSV DNA copy number, and (<strong>G</strong>) peak HSV DNA copy number. Model output with continuous release (82 HSV DNA particles per day) of virus randomly to 20 (yellow), 5 (light blue), and 1 (red) region underestimated (<strong>A</strong>) quantitative shedding frequency and episode, (<strong>B</strong>) rate, (<strong>D</strong>) Duration, (<strong>E</strong>) first HSV DNA copy number, and (<strong>G</strong>) peak HSV DNA copy number.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.037">http://dx.doi.org/10.7554/eLife.00288.037</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="fig" data-doi="10.7554/eLife.00288.038"><div id="fig6s2" class="fig-inline-img-set"><div class="elife-fig-image-caption-wrapper"><div class="fig-expansion"><div class="fig-inline-img"><a href="[graphic-elife00288fs010-large]" class="figure-expand-popup" title="Figure 6—figure supplement 2."><img data-img="[graphic-elife00288fs010-small]" src="[graphic-elife00288fs010-medium]" alt="Figure 6—figure supplement 2."/></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="[graphic-elife00288fs010-large-download]">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="[graphic-elife00288fs010-large]" target="_blank">Open in new tab</a></span></span><span class="fig-label">Figure 6—figure supplement 2.</span> <span class="caption-title">Dispersion of viral particles from neurons reproduced the full diversity of episode characteristics during simulations in which particles were released into a minority of modeled regions, provided that dispersion was random rather than clustered within a single geographic region.</span><p>White circles represent results from (<strong>A</strong>) 14,685 genital swabs and (<strong>B</strong>–<strong>G</strong>) 1020 shedding episodes from 531 study participants (Figure 4). The model simulations represented with colored bars in each panel continued until 1020 episodes were generated. Sampling occurred every 24 hr as in the clinical protocols. Model output with continuous release (82 HSV DNA particles per day) of virus to 50 randomly dispersed regions (brown) reproduced (<strong>A</strong>) quantitative shedding frequency and episode, (<strong>B</strong>) rate, (<strong>C</strong>) median initiation to peak slope and peak to termination slopes, (<strong>D</strong>) Duration, (<strong>E</strong>) first HSV DNA copy number, (<strong>F</strong>) last HSV DNA copy number, and (<strong>G</strong>) peak HSV DNA copy number. Model output with continuous release (82 HSV DNA particles per day) of virus to 50 clustered regions (green) underestimated (<strong>A</strong>) shedding frequency due only to (<strong>B</strong>) too low of an episode rate with daily sampling.</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.038">http://dx.doi.org/10.7554/eLife.00288.038</a></p><div class="sb-div caption-clear"></div></div></div></div></div></div></div></div>

        </div>
        <div id="s2-11" class="subsection">
            <h3>Determinants of intersubject variability in shedding and lesion rates</h3>
            <p>We next tested the model's ability to predict substantial variability in shedding, lesion, and episode rates in studies consisting of 30- to 60-day sampling periods (<a class="xref-bibr" href="#bib36">Wald et al., 1995</a>, <a class="xref-bibr" href="#bib37">1997</a>, <a class="xref-bibr" href="#bib38">2000</a>; <a class="xref-bibr" href="#bib5">Crespi et al., 2007</a>; <a class="xref-bibr" href="#bib18">Mark et al., 2008</a>). Even when parameter values were not varied, the stochastic model produced realistic outcomes (<a class="xref-table" href="#tbl4">Table 4</a>). Lesion and shedding frequency were variable with 30- and 60-day simulations, corresponding to previously published results (<a class="xref-bibr" href="#bib37">Wald et al., 1997</a>, <a class="xref-bibr" href="#bib38">2000</a>; <a class="xref-bibr" href="#bib18">Mark et al., 2008</a>). Simulations of 1 and 10 years, generated less varied outcomes (<a class="xref-table" href="#tbl4">Table 4</a>), suggesting that the random nature of episode initiation, along with short sampling periods may account for some shedding heterogeneity seen in clinical studies.</p>

            <div class="table-expansion" id="tbl4"><div class="table-caption"><span class="table-label">Table 4.</span> <p>Spatial model simulations that varied only according to duration of sampling (30 days, 60 days, 365 days, and 10 years)</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.039">http://dx.doi.org/10.7554/eLife.00288.039</a></p><div class="sb-div caption-clear"></div></div><table><thead><tr><td>Simulation duration</td><td></td><td>Percent of time with HSV DNA &gt; 150 copies per mL</td><td>Percent of time with lesions<a class="xref-table-fn" href="#tblfn1">*</a> present</td><td>Episodes per year</td><td>Lesions per year</td></tr></thead><tbody><tr><td rowspan="3">30 day</td><td>Mean</td><td>13.7</td><td>7.6</td><td>11</td><td>3.2</td></tr><tr><td>Median</td><td>3.4</td><td>0</td><td>12</td><td>0</td></tr><tr><td>Range</td><td>0–82.8</td><td>0–58.8</td><td>0–36.5</td><td>0–24.3</td></tr><tr><td rowspan="3">60 day</td><td>Mean</td><td>19.0</td><td>9.8</td><td>13.4</td><td>4.4</td></tr><tr><td>Median</td><td>18.6</td><td>3.4</td><td>12</td><td>6</td></tr><tr><td>Range</td><td>0–54.2</td><td>0–46.9</td><td>0–42.6</td><td>0–18</td></tr><tr><td rowspan="3">365 day</td><td>Mean</td><td>19.6</td><td>10.1</td><td>14.3</td><td>4.6</td></tr><tr><td>Median</td><td>19.9</td><td>9.9</td><td>14</td><td>4</td></tr><tr><td>Range</td><td>7.1–36.8</td><td>2.3–22</td><td>8–24</td><td>1–9</td></tr><tr><td rowspan="3">10 year</td><td>Mean</td><td>17.5</td><td>9.1</td><td>14.5</td><td>4.3</td></tr><tr><td>Median</td><td>17.6</td><td>9.0</td><td>14.5</td><td>4.2</td></tr><tr><td>Range</td><td>14.8–19.8</td><td>1.9–12.8</td><td>11.5–17.1</td><td>1.4–6</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="tblfn1"><label>*</label><p>Lesions were defined as &gt; 1 mm diameter ulcers.</p></li><li class="fn"><p>Sixty simulations were performed at each of the sampling durations. Within shorter sampling duration simulations, lesion, and shedding frequency varied significantly, while ranges narrowed with prolonged sampling.</p></li></ul></div></div>

            <p>To assess other possible hypotheses for contrasting shedding rates, we conducted another global sensitivity analysis in which 500 Monte Carlo simulations were conducted over a 10-year time frame (‘Methods’). Predictors of elevated shedding rate included high neuronal release rate of HSV-2 DNA, high CD8+ T-cell replication rate, high regional co-dependence of CD8+ T-cell density, low HSV-2 DNA replication rate in skin cells, and low free-viral decay rate (<a class="xref-table" href="#tbl3">Table 3</a>). The finding that parameter levels which would appear to favor immune control in the mucosa, in fact correlate with higher long-term shedding rates, highlight the complex nonlinear dynamics of the spatial model; parameters that favor lower amounts of viral replication in the short term promote a higher proportion of episodes that are contained rapidly by the immune system; CD8+ T-cell expansion is less substantial during these brief episodes. Therefore, episodes are more commonly initiated at lower CD8+ T-cell densities. As a result, conditions of high CD8+ T-cell replication rate and low HSV-2 DNA replication rate in keratinocytes tend to favor both a higher proportion of very brief episodes (&lt;1 day) and more severe episodes that spread to multiple regions. The overall impact is to increase the shedding rate.</p>
        </div>
    </div>
    <div class="section discussion" id="s3">
        <h2>Discussion</h2>
        <p>We used a strategy of intense sampling over intervals ranging from every 5 min to every 24 hr, to characterize temporal and spatial dynamics of genital HSV-2 replication. While we previously identified that a majority of HSV-2 reactivations last less than a day (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>), our current findings demonstrate that the cardinal feature of HSV-2 shedding is extraordinary episode heterogeneity. Longer and higher copy number episodes, which often manifest with genital lesions, are notable for multiple erratic peaks and wide viral dispersion. Viral loads are stable over periods of minutes but can change dramatically over hours. HSV-2 expansion and decay phases in mucosa are considerably more rapid and complex than those of HIV and hepatitis B, which have been characterized by sampling plasma (<a class="xref-bibr" href="#bib2">Ciupe et al., 2007</a>; <a class="xref-bibr" href="#bib22">Ribeiro et al., 2010</a>). To explain these findings, we developed a stochastic spatial model that accurately reproduces empirically derived episode rate, duration, early viral expansion, late viral decay rate, and viral re-expansion frequency, as well as serpiginous spatial distribution of viral ulcers during recurrences.</p>
        <p>Our findings redefine the pace of viral immune interaction within a single herpetic ulcer. HSV replication within epithelial cells, and spread by cell-to-cell transfer, seem to occur extremely rapidly and efficiently. The model predicts that &gt;10<sup>8</sup> viral DNA genomes can be produced in one ulcer, even though the immune response becomes dominant only ∼13.5 hr after HSV-2 detection. Routes of spread between tightly packed epithelial cells might include transit through tight cellular junctions (<a class="xref-bibr" href="#bib3">Collins and Johnson, 2003</a>; <a class="xref-bibr" href="#bib9">Farnsworth and Johnson, 2006</a>), distant propulsion when cells violently lyse, and homogenous mixing with susceptible cells as a result of high viral titers packed inside herpetic vesicles (<a class="xref-bibr" href="#bib30">Spruance et al., 1977</a>).</p>
        <p>The model is in agreement with mouse studies, which show that tissue-resident CD8+ T-cell density at the site of viral challenge dictates lesion severity (<a class="xref-bibr" href="#bib11">Gebhardt et al., 2009</a>; <a class="xref-bibr" href="#bib29">Shin and Iwasaki, 2012</a>), and further suggests that local CD8+ T-cell count supersedes all other parameters of viral replication and immunologic response in determining the extent of shedding during an episode. While common wisdom is that extended viral replication occurs due to delayed arrival of acquired immune cells to the site of viral pathogenesis, our observations suggest that the immune response in microregions of replication is extraordinarily rapid. Most shedding episodes are contained within hours, when &lt;10 cells are infected (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>). Yet, even if thousands of epithelial cells get infected in an ulcer, these cells appear to be eliminated in &lt;24 hr.</p>
        <p>Our model suggests that cell-free virus plays an important role in episode prolongation: cell-free viruses seed adjacent regions where T-cell density may be lower, leading to the formation of secondary foci of viral replication, which allows wide spread of virus across the genital tract. Thus, the spatial dynamic between virus and host defines episode morphology, and explains variability in shedding severity in an individual over periods of hours to weeks. These results suggest that studies of neutralizing and antibody-dependent cell-mediated cytotoxicity responses in localized mucosal areas are needed, and may explain why antiviral therapies which only impact intracellular replication and do not eliminate prolonged episodes.</p>
        <p>While the model successfully generates hypotheses to explain complex shedding patterns in humans, it also highlights the difficulty of predicting an individual's short-term shedding pattern, and in particular, the timing and morphology of their next episode. A heterogeneous spatial array of immune cell densities at episode onset may determine the complex trajectory of a shedding episode, but these starting conditions cannot be directly measured in humans. It is similarly not possible to know in which specific microregion an episode will initiate. When we simulated the model over 30-day intervals typically used in our clinical studies, there was heterogeneity in the shedding rate based solely on randomness of episode initiation and high variability of episode duration. As such, we did not employ well-validated methods to infer parameter values by fitting to an individual's nonlinear shedding data (<a class="xref-bibr" href="#bib12">Ionides et al., 2006</a>).</p>
        <p>On the other hand, the model does raise intriguing possible explanations for observed heterogeneity in long-term shedding behavior between infected persons (<a class="xref-bibr" href="#bib37">Wald et al., 1997</a>). Surprisingly, while enhanced immune function in the form of lower viral replication rate or viral infectivity has the predictable short-term effect of limiting episode duration, the model suggests that sustained higher T-cell CD8+ replication rate and lower viral replication rate might counterintuitively increase long-term shedding rates: these parameter conditions favor a higher proportion of rapidly contained episodes, which in turn lowers the degree of CD8+ expansion. Therefore, the CD8+ density at episode initiation is on average lower, which also allows for more frequent widely dispersed episodes. Because the overall stability of parameter values over time cannot be easily measured, and because it is unknown if the model's parameters behave independently or in a colinear fashion, these sensitivity analysis predictions are relevant for hypothesis generation only. An area of prospective research is to identify which virological and immune factors account for phenotypic variability between infected persons.</p>
        <p>For the model to precisely recapitulate episode rate, it is necessary to assume a nearly constant slow drip of HSV-2 from neurons (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>). The stochastic model and spatial shedding data both suggest that there is diffuse dispersal of virus from neurons into genital epithelia, rather than focal reactivations only within certain microregions. Such a strategy allows HSV-2 to randomly seed regions where immune cell decay from previous reactivations may allow a high-titer episode to initiate. This does not imply that latency is constantly bypassed within all neurons. Rather, we favor the idea that the vast majority of HSV within ganglia is maintained in a low copy number latent state (<a class="xref-bibr" href="#bib39">Wang et al., 2005</a>; <a class="xref-bibr" href="#bib35">Verjans et al., 2007</a>), while a minority reactivates albeit on a frequent basis. More experimental work is needed to precisely define the true mechanisms that form a balance between HSV latency and replication within ganglia.</p>
        <p>Our results have interesting implications for HSV-2 transmission during coitus. Most transmissions occur when an infected person is asymptomatic (<a class="xref-bibr" href="#bib19">Mertz et al., 1985</a>, <a class="xref-bibr" href="#bib20">1998</a>), a finding that can be interpreted within the context of our model's output. Secondary seeding starts to predictably occur in modeled episodes when viral loads in a single ulcer exceed 10<sup>6</sup> HSV DNA copies. At this viral load, there are roughly 1000 dead cells, which is not enough for a microulcer to become visible to the naked eye. Therefore, HSV-2 can efficiently seed new regions across the genital tract leading to prolonged high copy number episodes even when shedding is not associated with lesions.</p>
        <p>The goal of this study was to design a model that is realistic and adequately complex to recreate our stringently defined dataset, but simple enough to have identifiable parameters. Hence, one limitation is that the model underestimates immunologic complexity. CD8+ T-cell response is assumed to be a surrogate of the entire immune response. Other key cellular subsets such as CD4+ lymphocytes, natural killer cells, and dendritic cells are not included. Lack of inclusion of innate immunity is another shortcoming as interferon type 1 is present early during lesions (<a class="xref-bibr" href="#bib31">Spruance et al., 1982</a>), and contributes to episode containment. Our model generates the hypothesis that chronic HSV-2 infection leads to a dynamic and spatially heterogeneous state of immune readiness. However, the interacting features of host response are not addressed and require further investigation using human and animal models of infection.</p>
        <p>For convenience, we divided the genital tract into 300 regions. Simulations with 200–400 regions provided similar fit to the data (not shown). To model spatial spread most completely would require following each cell in the genital tract with an agent-based model as well as accurate spatial elaboration of the genital anatomy (<a class="xref-bibr" href="#bib1">Bauer et al., 2009</a>): this would entail great computational cost and inclusion of many unknown parameters. The high number of fitting points in Cohort E and limited number of unknown parameters in our model decreases but does not eliminate the possibility that the model has multiple solutions. If swabs only remove a fixed proportion of free-viral DNA, then it is possible that we are underestimating the genital tract viral load. Based on the complex nature of the observational data, the use of a stochastic simulation model, and the inability to manipulate experimental conditions in our longitudinal studies of human shedding, our parameter values are estimates rather than precise solutions. We believe that the most critical hypotheses generated from this study are qualitative, and are relevant even if certain parameter values were not precisely identified.</p>
        <p>In summary, our data suggest intensely localized competition between HSV-2 and the immune system in genital mucosa. Our model raises the possibility that by rapidly spreading between epidermal cells, HSV-2 ensures high viral production within a single ulcer, despite an intense local immune response. Cell-free HSV-2 may prolong episodes via spread to contiguous sites where immune cell density is lower. Each episode is contained by the mucosal immune system, and HSV-2 is often asymptomatic in immunocompetent hosts. Viral strategies that lead to repeated episodes, rapid episode expansion and frequent re-expansion, allow for high shedding frequency, thereby enhancing transmission and disease manifestations. Interventions that target spread of virus between epidermal cells in a single ulcer, and to new regions of the genital tract, will be of value in controlling HSV-2 infection.</p>
    </div>
    <div class="section materials-methods" id="s4">
        <h2>Materials and methods</h2>
        <div id="s4-1" class="subsection">
            <h3>Study procedures</h3>
            <p>Study participants signed informed consent for the human experimentation described in this paper. All potential risks of being involved in the protocols, including discomfort, bleeding and infection, were explained to all participants in detail. The consent process including agreeing to have the experimental data published. Study subjects were reimbursed for clinic visits and the considerable time dedicated to each of the protocols. The University of Washington institutional review board approved all study protocols described in this study.</p>
            <p>HSV-2 seropositive participants performed swabs of the genital tract, which were subsequently processed for quantitative HSV DNA PCR using polymerase chain reaction (PCR) (<a class="xref-bibr" href="#bib37">Wald et al., 1997</a>; <a class="xref-bibr" href="#bib16">Magaret et al., 2007</a>; <a class="xref-bibr" href="#bib18">Mark et al., 2008</a>). Detection of HSV DNA was performed using a well-validated collection and detection method (<a class="xref-bibr" href="#bib16">Magaret et al., 2007</a>). Swabs of genital secretions were placed into vials containing 1 ml of PCR transport medium and refrigerated until laboratory processing. HSV DNA was detected using a quantitative PCR assay, and was expressed as copies per milliliter of medium. The PCR assay uses type-common primers to the HSV gene encoding glycoprotein B. An internal control was included to ensure that HSV-negative swabs were not due to inhibition. Laboratory personnel were blinded to clinical data.</p>
        </div>
        <div id="s4-2" class="subsection"><h3>Analysis of shedding data</h3>
            <p>We designed 5 separate cohorts, which used common sample protocols and virological assay but differed according to how frequently patients swabbed their genital tract and according to location of sampling (<a class="xref-table" href="#tbl1">Table 1</a>). Cohort E was used for fitting the model (<a class="xref-bibr" href="#bib28">Schiffer et al., 2011</a>): participants swabbed their entire genital tract every 24 hr whether lesions were present or not for a minimum of 30 days. We used eight summary measures (five frequency histograms and three median measures, <a class="xref-fig" href="#fig4">Figure 4</a>), which included a total of 42 data bins, to describe fundamental characteristics of shedding episodes. Three histograms described frequency of different HSV DNA copy number values within logarithmically defined strata (peak, first, and last positive swab within an episode, respectively). Two frequency histograms described episode duration, and frequency of total swabs within logarithmically defined strata. Duration of episodes was measured with interval censoring (whereby episodes of unknown duration due to initiation before day 0 of the shedding protocol or termination following the end of the swabbing period were excluded) and no interval censoring (whereby these episodes were included) (<a class="xref-fig" href="#fig4">Figure 4D</a>). Median measures included regression slopes from initiation to episode peak and episode peak to termination, as well as episode frequency (<a class="xref-fig" href="#fig4">Figure 4B,C</a>).</p>
            <p>Cohort C was similar to Cohort E, except participants swabbed every 6 hr (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>), allowing for more precise measures of episode rate, early episode expansion rate, and late episode decay rate. Cohort C was not used for direct model fitting because the number of subjects was much larger in Cohort E. During Cohort C, episodes of age 0–6 hr were captured with the first positive swab value: the median episode within the 2 datasets was captured 3 hr (0.125 days) after shedding episode initiation. We therefore divided median first positive swab copy number by 0.125 to estimate the median early expansion slope. We used an equivalent approach using last positive copy number from episodes to estimate a late decay slope.</p>
            <p>We calculated initiation to peak slopes in Cohorts E and C, with the assumption that episodes initiated 12 and 3 hr before the first positive swab, respectively, and fit a regression model from this point to episode peak. We calculated peak to termination slopes in Cohorts E and C, with the assumption that episodes ended 12 and 3 hr after the last positive swab, respectively, and fit a regression model from this episode peak to this point. The narrow interval of swabs from Cohort C was likely to represent a more accurate estimate of expansion and decay kinetics. However, median values from Cohort E (<a class="xref-fig" href="#fig4">Figure 4B</a>) were useful as fitting measures because we also sampled daily when fitting the model.</p>
            <p>As detailed in <a class="xref-table" href="#tbl1">Table 1</a>, we performed two shedding protocols (Cohorts A and B) when genital lesions were present. To demonstrate spatial analysis of shedding (Cohort D), study participants had a daily detailed genital examination by an experienced clinician. If lesions were present, the lesion location was recorded on genital diagrams. Twenty-three distinct sites were swabbed at each daily visit (<a class="xref-fig" href="#fig2">Figure 2A</a>). Care was taken to avoid contamination from other anatomic areas during sampling (<a class="xref-bibr" href="#bib33">Tata et al., 2010</a>).</p>
        </div>
        <div id="s4-3" class="subsection">
            <h3>Analysis of biopsy data</h3>
            <p>We defined spatiotemporal dynamics of HSV-2–specific lymphocyte response to a genital recurrence with in situ staining of genital skin biopsy with HSV-2 antigen-specific quantum dot multimers (<a class="xref-bibr" href="#bib40">Zhu et al., 2007</a>, <a class="xref-bibr" href="#bib41">2009</a>). We then examined biopsy specimens with confocal microscopy. CD8+ lymphocytes appeared at the precise site of HSV release from the neuron termini, the dermal/epidermal junction. Previous studies indicated that at least 10% of CD8+ T cells detected in tissue at the dermal–epidermal junction are HSV-2 specific and that this proportion remains constant as CD8+ T-cell levels fluctuate (<a class="xref-bibr" href="#bib40">Zhu et al., 2007</a>). Therefore, we used data derived from in situ staining of CD8+ T cells to define relative densities of HSV-2–specific CD8+ T cells in mucosal tissue. In previous studies, absolute number of CD8+ lymphocytes was measured per square millimeter of genital skin. These measures are underestimates for total number of CD8+ T cells present in the entire lesion area because most lesions are &gt;1 mm<sup>2</sup>. Values of sequential CD8+ T-cell densities gathered every 2 weeks therefore estimate relative intensity of the immune response at different time points.</p>
        </div>
        <div id="s4-4" class="subsection">
            <h3>Mathematical model fitting to empirical data</h3>
            <p>All preliminary simulations were performed using Berkley Madonna, while C++ was used for all spatial modeling, high throughput simulations, parameter fitting, sensitivity analyses, and video generation. The goal of our models was to reproduce the 42 data bins within the 8 summary measures of HSV-2 shedding from Cohort E. To this end, we iteratively developed four models with competing equations (<a class="xref-table" href="#tbl5">Table 5</a>). We solved models stochastically due to the random nature of shedding episode initiation and clearance, and to account for the frequent presence of low numbers of infected cells: at each time step, integer values for equation terms were drawn randomly from binomial distributions. To assess the degree of fit between models, numerical output and empirical data required prolonged simulations to minimize fluctuations in output due to stochastic effect. For each fitting attempt, we ran the model until 500–1000 episodes were generated. While state variables were updated at a narrow time interval (0.001 days), we assembled the modeled data exactly as it was gathered in the clinical protocols by sampling every 24 hr.</p>

            <div class="table-expansion" id="tbl5"><div class="table-caption"><span class="table-label">Table 5.</span> <p>Mathematical models of HSV-2 pathogenesis</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.040">http://dx.doi.org/10.7554/eLife.00288.040</a></p><div class="sb-div caption-clear"></div></div><table><thead><tr><td>Model</td><td>Equations (additions to previous model are denoted in bold)</td><td>Variables (model fitting variable)</td><td>New features</td></tr></thead><tbody><tr><td>1 (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>; <a class="xref-bibr" href="#bib26">Schiffer et al., 2010</a>)</td><td><span class="inline-formula"><span class="mathjax mml-math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">S</mi><mo>=</mo><mo>[</mo><mo>λ</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><mi mathvariant="normal">V</mi><mo>)</mo><mo>−</mo><mo>(</mo><mo>β</mo><mi>i</mi><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">I</mi><mo>=</mo><mo>[</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>+</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">a</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">f</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">E</mi><mo>=</mo><mo>[</mo><mo>(</mo><mi mathvariant="normal">F</mi><mo>(</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>×</mo><mi>θ</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi>δ</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>=</mo><mo>[</mo><mo>(</mo><mi mathvariant="normal">p</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">c</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>=</mo><mo>[</mo><mi>φ</mi><mo>−</mo><mo>(</mo><mi mathvariant="normal">c</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mo>&thinsp;</mo><mi mathvariant="normal">F</mi><mo>(</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>=</mo><mi mathvariant="normal">I</mi><mo>/</mo><mo>(</mo><mi mathvariant="normal">I</mi><mo>+</mo><mi mathvariant="normal">r</mi><mo>)</mo></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">V</mi><mrow><mtext>tot</mtext></mrow></msub><mo>=</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>+</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>λ</mo><mo>=</mo><mi>d</mi><mo>(</mo><mi mathvariant="normal">S</mi><mo>−</mo><mtext>S0</mtext><mo>)</mo></mtd></mtr></mtable></math></span></span></td><td>S, I, E, V<sub>i</sub> V<sub>neu</sub>,</td><td>—</td></tr><tr><td>2</td><td><span class="inline-formula"><span class="mathjax mml-math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">S</mi><mo>=</mo><mo>[</mo><mo>λ</mo><mo>−</mo><mo>(</mo><msub><mo mathvariant="bold">β</mo><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">×</mo><mi mathvariant="normal">S</mi><mo mathvariant="bold">×</mo><mi mathvariant="normal">V</mi><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo mathvariant="bold">β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">I</mi><mo>=</mo><mo>[</mo><mo mathvariant="bold">(</mo><msub><mo mathvariant="bold">β</mo><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">×</mo><mi mathvariant="normal">S</mi><mo mathvariant="bold">×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>+</mo><mo>(</mo><msub><mo mathvariant="bold">β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">a</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">f</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">E</mi><mo>=</mo><mo>[</mo><mo>(</mo><mi mathvariant="normal">F</mi><mo>(</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>×</mo><mi>θ</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi>δ</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>=</mo><mo>[</mo><mi>φ</mi><mo>−</mo><mo>(</mo><mi mathvariant="normal">c</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo mathvariant="bold">β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo mathvariant="bold">Δ</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">=</mo><mo mathvariant="bold">[</mo><mo mathvariant="bold">(</mo><mi mathvariant="bold">p</mi><mo mathvariant="bold">×</mo><mi mathvariant="normal">I</mi><mo mathvariant="bold">)</mo><mo mathvariant="bold">−</mo><mo mathvariant="bold">(</mo><mi mathvariant="normal">a</mi><mo mathvariant="bold">×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">)</mo><mo mathvariant="bold">−</mo><mo mathvariant="bold">(</mo><msub><mo mathvariant="bold">β</mo><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">×</mo><mi mathvariant="normal">S</mi><mo mathvariant="bold">×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">)</mo><mo mathvariant="bold">]</mo><mrow><mo>&thinsp;</mo></mrow><mo mathvariant="bold">Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo mathvariant="bold">Δ</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo mathvariant="bold">=</mo><mo mathvariant="bold">[</mo><mo mathvariant="bold">(</mo><mi mathvariant="normal">a</mi><mo mathvariant="bold">×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">)</mo><mo mathvariant="bold">−</mo><mo mathvariant="bold">(</mo><mi mathvariant="normal">c</mi><mo mathvariant="bold">×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo mathvariant="bold">)</mo><mo mathvariant="bold">]</mo><mrow><mo>&thinsp;</mo></mrow><mo mathvariant="bold">Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mo>&thinsp;</mo><mi mathvariant="normal">F</mi><mo>(</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>=</mo><mi mathvariant="normal">I</mi><mo>/</mo><mo>(</mo><mi mathvariant="normal">I</mi><mo>+</mo><mi mathvariant="normal">r</mi><mo>)</mo></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">V</mi><mrow><mtext mathvariant="bold">tot</mtext></mrow></msub><mo mathvariant="bold">=</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">+</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo mathvariant="bold">+</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext mathvariant="bold">neu</mtext></mrow></msub></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>λ</mo><mo>=</mo><mi mathvariant="normal">d</mi><mo>(</mo><mi mathvariant="normal">S</mi><mo>−</mo><mtext>S0</mtext><mo>)</mo></mtd></mtr></mtable></math></span></span></td><td>S, I, E, V<sub>i</sub>, V<sub>neu</sub>, <strong>(V<sub>e</sub>)</strong></td><td>Cell-free and cell-associated particles</td></tr><tr><td>3</td><td><span class="inline-formula"><span class="mathjax mml-math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">S</mi><mo mathvariant="bold">(</mo><mi mathvariant="bold">i</mi><mo mathvariant="bold">…</mo><mn mathvariant="bold">300</mn><mo mathvariant="bold">)</mo><mo>=</mo><mo>=</mo><mo>[</mo><mo>λ</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo mathvariant="bold">(</mo><msub><mo mathvariant="bold">β</mo><mi mathvariant="normal">e</mi></msub><mo mathvariant="bold">×</mo><mi mathvariant="normal">S</mi><mo mathvariant="bold">×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo mathvariant="bold">)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">I</mi><mo mathvariant="bold">(</mo><mi mathvariant="bold">i</mi><mo mathvariant="bold">…</mo><mn mathvariant="bold">300</mn><mo mathvariant="bold">)</mo><mo>=</mo><mo>[</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>+</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo mathvariant="bold">(</mo><msub><mo mathvariant="bold">β</mo><mi mathvariant="normal">e</mi></msub><mo mathvariant="bold">×</mo><mi mathvariant="normal">S</mi><mo mathvariant="bold">×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext mathvariant="bold">etot</mtext></mrow></msub><mo mathvariant="bold">)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">a</mi><mo>×</mo><mi mathvariant="normal">l</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">f</mi><mo>×</mo><mi mathvariant="normal">l</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">S</mi><mo mathvariant="bold">(</mo><mi mathvariant="bold">i</mi><mo mathvariant="bold">…</mo><mn mathvariant="bold">300</mn><mo mathvariant="bold">)</mo><mo>=</mo><mo>[</mo><mo>(</mo><mi mathvariant="normal">F</mi><mo>(</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>×</mo><mi>θ</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi>δ</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi>t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo mathvariant="bold">(</mo><mi mathvariant="bold">i</mi><mo mathvariant="bold">…</mo><mn mathvariant="bold">300</mn><mo mathvariant="bold">)</mo><mo>=</mo><mo>[</mo><mi>φ</mi><mo>−</mo><mo>(</mo><mi mathvariant="normal">c</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi>t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo mathvariant="bold">(</mo><mi mathvariant="bold">i</mi><mo mathvariant="bold">…</mo><mn mathvariant="bold">300</mn><mo mathvariant="bold">)</mo><mo>=</mo><mo>[</mo><mo>(</mo><mi>p</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">a</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi>i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>Δ</mo><mi>t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo>(</mo><mi>i</mi><mo>…</mo><mn>300</mn><mo>)</mo><mrow><mo>&nbsp;</mo></mrow><mo>=</mo><mrow><mo>&nbsp;</mo></mrow><mo>[</mo><mo>(</mo><mi mathvariant="normal">a</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">c</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo>)</mo><mo>]</mo><mo>Δ</mo><mi>t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mo>&thinsp;</mo><mi mathvariant="normal">F</mi><mo>(</mo><mi mathvariant="normal">I</mi><mo>)</mo><mrow><mo>&nbsp;</mo></mrow><mo>=</mo><mi mathvariant="normal">I</mi><mo>/</mo><mo>(</mo><mi mathvariant="normal">I</mi><mo>+</mo><mi mathvariant="normal">r</mi><mo>)</mo></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">I</mi><mrow><mtext mathvariant="bold">tot</mtext></mrow></msub><mo mathvariant="bold">=</mo><msub><mi mathvariant="normal">I</mi><mn mathvariant="bold">1</mn></msub><mo mathvariant="bold">+</mo><msub><mi mathvariant="normal">I</mi><mn mathvariant="bold">2</mn></msub><mo mathvariant="bold">+</mo><mo mathvariant="bold">…</mo><mo mathvariant="bold">+</mo><msub><mi mathvariant="normal">I</mi><mrow><mtext mathvariant="bold">e300</mtext></mrow></msub></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">V</mi><mrow><mtext mathvariant="bold">etot</mtext></mrow></msub><mo mathvariant="bold">=</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">e</mi><mn mathvariant="bold">1</mn></mrow></msub><mo mathvariant="bold">+</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">e</mi><mn mathvariant="bold">2</mn></mrow></msub><mo mathvariant="bold">+</mo><mo mathvariant="bold">…</mo><mo mathvariant="bold">+</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">e</mi><mn mathvariant="bold">300</mn></mrow></msub></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">V</mi><mrow><mtext mathvariant="bold">itot</mtext></mrow></msub><mo mathvariant="bold">=</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">i</mi><mn mathvariant="bold">1</mn></mrow></msub><mo mathvariant="bold">+</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">i</mi><mn mathvariant="bold">2</mn></mrow></msub><mo mathvariant="bold">+</mo><mo mathvariant="bold">…</mo><mo mathvariant="bold">+</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">i</mi><mn mathvariant="bold">300</mn></mrow></msub></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>λ</mo><mo>=</mo><mi>d</mi><mo>(</mo><mi mathvariant="normal">S</mi><mo>−</mo><mtext>S0</mtext><mo>)</mo></mtd></mtr></mtable></math></span></span></td><td>S, I, E, V<sub>neu</sub>, V<sub>itot</sub>, <strong>(V<sub>etot</sub>)</strong></td><td>Concurrent plaques from cell-free particles</td></tr><tr><td>4*</td><td><span class="inline-formula"><span class="mathjax mml-math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mo>•</mo><mo>&thinsp;</mo><mtext>S0</mtext><mo>=</mo><mtext>1.67 e5 per region</mtext></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">S</mi><mo>(</mo><mi>i</mi><mo>…</mo><mn mathvariant="bold">300</mn><mo>)</mo><mo>=</mo><mo>[</mo><mi>λ</mi><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">e</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">I</mi><mo>(</mo><mi>i</mi><mo>…</mo><mn mathvariant="bold">300</mn><mo>)</mo><mo>=</mo><mo>[</mo><mo>(</mo><msub><mi>β</mi><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>+</mo><mo>(</mo><msub><mi>β</mi><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>+</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">e</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext mathvariant="bold">eadj</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">a</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">f</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">E</mi><mo>(</mo><mi>i</mi><mo>…</mo><mn mathvariant="bold">300</mn><mo>)</mo><mo>=</mo><mo>[</mo><mo>(</mo><mi mathvariant="normal">F</mi><mo>(</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>×</mo><mi>θ</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi>δ</mi><mo>×</mo><mi mathvariant="normal">E</mi><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>(</mo><mi>i</mi><mo>…</mo><mn mathvariant="bold">300</mn><mo>)</mo><mo>=</mo><mo>[</mo><mi>φ</mi><mo>−</mo><mo>(</mo><mi mathvariant="normal">c</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mrow><mtext>neu</mtext></mrow></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>(</mo><mi>i</mi><mo>…</mo><mn mathvariant="bold">300</mn><mo>)</mo><mo>=</mo><mo>[</mo><mo>(</mo><mi>p</mi><mo>×</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">a</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>−</mo><mo>(</mo><msub><mo>β</mo><mi mathvariant="normal">i</mi></msub><mo>×</mo><mi mathvariant="normal">S</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo>(</mo><mi>i</mi><mo>…</mo><mn mathvariant="bold">300</mn><mo>)</mo><mo>=</mo><mo>[</mo><mo>(</mo><mi mathvariant="normal">a</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">i</mi></msub><mo>)</mo><mo>−</mo><mo>(</mo><mi mathvariant="normal">c</mi><mo>×</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mo>)</mo><mo>]</mo><mrow><mo>&thinsp;</mo></mrow><mo>Δ</mo><mi mathvariant="normal">t</mi></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><mo>λ</mo><mo>=</mo><mi>d</mi><mo>(</mo><mi mathvariant="normal">S</mi><mo>−</mo><mtext>S0</mtext><mo>)</mo></mtd></mtr><mtr><mtd><mo>•</mo><mo>&thinsp;</mo><mi mathvariant="normal">F</mi><mo>(</mo><mi mathvariant="normal">I</mi><mo>)</mo><mo>=</mo><mtext>I/</mtext><mo>(</mo><mi mathvariant="normal">I</mi><mo>+</mo><mi>r</mi><mo>)</mo></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">V</mi><mrow><mtext mathvariant="bold">eadj</mtext></mrow></msub><mo mathvariant="bold">=</mo><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">e</mi></msub><mtext mathvariant="bold">from 6 adjacent regions</mtext></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">I</mi><mrow><mtext>tot</mtext></mrow></msub><mo>=</mo><msub><mi mathvariant="normal">I</mi><mn>1</mn></msub><mo>+</mo><msub><mi mathvariant="normal">I</mi><mn>2</mn></msub><mo>+</mo><mo>…</mo><mo>+</mo><msub><mi mathvariant="normal">I</mi><mrow><mn mathvariant="bold">300</mn></mrow></msub></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">V</mi><mrow><mtext>etot</mtext></mrow></msub><mo>=</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">e</mi><mn>1</mn></mrow></msub><mo>+</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">e</mi><mn>2</mn></mrow></msub><mo>+</mo><mo>…</mo><mo>+</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">e</mi><mn mathvariant="bold">300</mn></mrow></msub></mtd></mtr><mtr><mtd><mo>•</mo><mrow><mo>&thinsp;</mo></mrow><msub><mi mathvariant="normal">V</mi><mrow><mtext>itot</mtext></mrow></msub><mo>=</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">i</mi><mn>1</mn></mrow></msub><mo>+</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">i</mi><mn>2</mn></mrow></msub><mo>+</mo><mo>…</mo><mo>+</mo><msub><mi mathvariant="normal">V</mi><mrow><mi mathvariant="normal">i</mi><mn mathvariant="bold">300</mn></mrow></msub></mtd></mtr></mtable></math></span></span></td><td>S, I, E, V<sub>neu</sub>, V<sub>itot</sub>, <strong>(V<sub>etot</sub>)</strong></td><td>Spatial model</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn"><p>*Model 4 has a parameter of regional CD8+ co-dependence (ρ) within each plaque-forming region. At episode onset within a region, the CD8+ density is adjusted to infer the spatial co-dependence of CD8 density from surrounding regions: E<sub>i</sub> (time + 0.001) = (E<sub>i</sub> <em>×</em> (1 − ρ)) + (E<sub>avg</sub> × ρ) where E<sub>avg</sub> is average of E from the 6 surrounding regions.</p></li><li class="fn"><p>Models are described in the ‘Methods’. Units and values of each parameter in the optimized model are listed in <span class="xref-table">Table 2</span>. Variables include: S (susceptible skin cells), I (infected skin cells), E (CD8+ T cells), V<sub>i</sub> (cell-associated HSV DNA particles), and V<sub>e</sub> (cell-free HSV DNA particles).</p></li></ul></div></div>

            <p>For each of the four competing mathematical models (<a class="xref-table" href="#tbl5">Table 5</a>), we performed repeated simulations while adjusting and narrowing parameter values (see ‘Parameter value search’ below) until a weighted least squares approach no longer improved fit to the 8 frequency histograms from Cohort E. For each model fitting simulation, we assigned each of the summary measures (1. episode rate, 2. episode duration, 3. median initiation to peak slope, 4. median peak to termination slope, 5. first positive copy number of episodes, 6. last positive copy number of episodes, 7. peak positive copy number of episodes, and 8. quantitative shedding) a weighting factor to ensure that each summary measure carried an equivalent weight. First, using the empirical data (colored bars, <a class="xref-fig" href="#fig4">Figure 4</a>), the mean value of bins within each of the five histograms (1. episode duration, 2. first positive copy number of episodes, 3. last positive copy number of episodes, 4. peak positive copy number of episodes, and 5. quantitative shedding) was calculated; the inverse square of this value was then used to generate an ‘initial weighting factor’, which was then divided by the number of bins within the histogram such that each bin was assigned a ‘bin weighting factor’. The three median measures (1. episode rate, 2. median initiation to peak slope, and 3. median peak to termination slope) only contained one bin such that the initial weighting factors were equal to the bin weighting factors.</p>
            <p>We calculated a ‘fitting score’ for every simulation of the model. For each bin, the difference between the empirical data and model output was squared and multiplied by the bin weighting factor for the bin, to arrive at a bin score. For the episode duration histogram, if the modeled value fell between the interval censored and noninterval censored empirical values, then the difference was assumed to be zero. Each simulation was given a fitting score equal to the sum of these 42 ‘bin scores’ with a lower score representing better model fit. Using an ad hoc approach, we compared fitting scores with repeated automations of the same model but different parameter sets, to arrive at the optimal set of values for each model; once a model's scores no longer improved, we assigned the model a best fitting score. Best fitting scores were compared between our four models (<a class="xref-table" href="#tbl6">Table 6</a>) in order to select the superior model.</p>

            <div class="table-expansion" id="tbl6"><div class="table-caption"><span class="table-label">Table 6.</span> <p>Model fit to Cohort E</p><p><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.00288.041">http://dx.doi.org/10.7554/eLife.00288.041</a></p><div class="sb-div caption-clear"></div></div><table><thead><tr><td rowspan="2">Summary measure</td><td colspan="8">Summed criteria scores</td><td rowspan="2">Best fitting score</td><td rowspan="2">AIC</td></tr><tr><td>Shedding frequency</td><td>Episode duration</td><td>First positive swab</td><td>Last positive swab</td><td>Peak positive swab</td><td>Median expansion</td><td>Median decay</td><td>Episode frequency</td></tr></thead><tbody><tr><td>Model 1</td><td>2.13</td><td>5.59</td><td>0.05</td><td>0.08</td><td>0.24</td><td>0.07</td><td>0.43</td><td>0.01</td><td>8.61</td><td>−50</td></tr><tr><td>Model 2</td><td>1.26</td><td>8.33</td><td>0.38</td><td>0.34</td><td>0.22</td><td>0.04</td><td>0.09</td><td>0.03</td><td>10.69</td><td>−41</td></tr><tr><td>Model 3</td><td>0.25</td><td>3.75</td><td>0.31</td><td>0.22</td><td>0.61</td><td>0.05</td><td>0.50</td><td>0.22</td><td>5.92</td><td>−62</td></tr><tr><td>Model 4 solved for 10 parameters</td><td>0.25</td><td>0.13</td><td>0.29</td><td>0.15</td><td>0.09</td><td>0.01</td><td>0.01</td><td>0.03</td><td>0.96</td><td>−139</td></tr><tr><td>Model 4 solved for 5 parameters</td><td>0.39</td><td>0.32</td><td>0.44</td><td>0.21</td><td>0.09</td><td>0.04</td><td>0</td><td>0.18</td><td>1.67</td><td>−125</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn"><p>Summed criteria scores measure the degree of fit for each model according the eight individual shedding episode features using a weighted sum of squares. Model 4 is the spatial model. Models 1–3 are described in the ‘Methods’. Best fitting score is a sum of all summed criteria scores for a particular model with lower scores indicating better fit. AIC: Akaike information criteria with lower scores indicating better fit.</p></li></ul></div></div>

            <p>We complemented the best fitting score approach by using ‘Akaike information criteria (AIC) scores’ as a standardized measure of model agreement with data. The AIC penalizes for extra parameters but rewards better model fit to the data. For our model, the AIC was calculated as AIC = N × ln (RSS/N) + 2k, where N = number of fitting points, k = number of parameters in the model, and RSS is the residual sum of squares, which we equated to the best fitting score.</p>
            <p>To characterize how well each of the four models reproduced individual fitting criteria, we generated a standardized grading system that accounted for the unique nature of our dataset (<a class="xref-table" href="#tbl6">Table 6</a>). For every fitting simulation, we summed the bin scores in each of the eight histograms into individual ‘summed criteria scores’. We observed that summed criteria scores &gt;1 resulted in significant disparity between the empirical and modeled data, while scores between 0.5 and 1 signified only moderate differences. Finally, if sums of the scores within a histogram amounted to &lt;0.5, then there was close similarity between the modeled data. For each simulation, if a summed criteria score exceeded 1, then we labeled the simulation a ‘poor’ fit to the summary measure. If a summed criteria score exceeded 5, then we labeled the simulation a ‘very poor’ fit. If a summed criteria score exceeded 0.5 but was &lt;1.0, we labeled the simulation a ‘fair’ fit, while scores &lt;0.5 were considered a ‘good’ fit. Using this approach, we specified which facets of shedding certain models could not replicate. We used summed criteria scores to provide ranges for optimized parameter values (<a class="xref-table" href="#tbl2">Table 2</a>).</p>
            <div id="s4-4-1" class="subsection">
                <h4>Mathematical model 1: baseline model of viral replication and CD8+ T-cell response</h4>
                <p>The initial model that we tested in our analysis (Model 1, <a class="xref-table" href="#tbl5">Table 5</a>) has been described (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>, <a class="xref-bibr" href="#bib26">2010</a>). The HSV-2 replication cycle in the genital tract and immunologic response to HSV-2–infected cells can be referenced to each of the model's parameters. The model assumes chronic infection. During the ‘latent phase’ of chronic genital HSV infection, virus is maintained in a dormant state in the sacral ganglia. During ‘reactivation’, virions are released from neuronal endings at the dermal–epidermal junction at a certain rate (ϕ). Released HSV-2 survives in the genital tract for a defined duration (1/<em>c</em>), during which time it can transfer from neurons and infect local epithelial cells. Viral infectivity (1/[β<sub>i</sub> × S]) is roughly defined as the number of viruses that are needed to infect one epithelial cell per day assuming the constant presence of viruses and a full complement of susceptible cells.</p>
                <p>For all of the models in this manuscript, we assumed some degree of target cell limitation, by virtue of susceptible cells decreasing relative to total number of cells with time. In one sense, target cells appear to be limitless in the genital tract: visualization of skin during lesions reveals a high ratio of uninfected to infected keratinocytes in genital skin (<a class="xref-fig" href="#fig2">Figure 2C</a>); even in severely immunocompromised hosts with severe infections, ulcers never involve the entire genital tract and do not spread more than a few centimeters down the thigh or up the stomach or back. Yet, spread of virus from the cytosol of one cell to another is constrained by the number of cellular contacts of an infected cell: as an ulcer spreads in radius, an increasing proportion cells in contact with an infected cell are already infected. This effect may decrease average infectivity of an infected cell as ulcer radius expands. Therefore, the infectivity term does decrease with time in our model according to number of cells that remain susceptible.</p>
                <p>We assumed that if an epithelial cell becomes infected, then it will die via direct lysis if it survives long enough to become packed with viruses (lifespan = 1/<em>a</em>), or via CD8+ lymphocyte–mediated killing (lifespan = 1/[<em>f</em> × E]). Lymphocyte killing efficiency (<em>f</em>) is defined as the number of infected cells cleared by one CD8+ lymphocyte cell per day in vivo. If an epithelial cell evades CD8+ lymphocyte mediated killing for the entire duration of cellular infection, it will produce a total of <em>p/a</em> viruses: <em>p</em> is the rate of viruses produced by an infected cell per day. For each fitting simulation, we included a minimum value of E within each region to avoid viral loads &gt;10<sup>10</sup> HSV DNA copies, which are never present clinically.</p>
                <p>HSV-2 reactivation that results in mucosal HSV shedding can be subclinical or clinical. Clinical shedding is termed a ‘recurrence’ and is associated with sporadic crops of vesicles, which form due to the death of closely congregated epithelial cells. Vesicles progress to ulcers and then undergo healing due to regrowth of susceptible cells: we assume that regrowth occurs according to a growth rate, λ or <em>d ×</em> (S0 − S) with growth limited by S0, the carrying capacity of the system. ‘Subclinical shedding episodes’ are common during chronic HSV-2 infection and are defined by local detection of virus either by culture or PCR, in the absence of a clinically apparent genital lesion or symptoms (<a class="xref-bibr" href="#bib18">Mark et al., 2008</a>).</p>
                <p>The formation of a genital lesion is accompanied by rapid accumulation of localized CD8+ lymphocytes at the dermal–epidermal junction at a peak rate (θ), followed by slow decay of these cells over a period of months after lesion healing (lifespan = 1/δ). The CD8+ replication rate in our model is saturated at θ, and θ/2 occurs when infected cells are equal in number to parameter <em>r</em>, which represents how many epithelial cells need to be infected before half-maximal CD8+ expansion. Therefore, parameter <em>r</em> defines how rapidly immune cells recognize and respond to viral antigens on the surface of infected cells. We have previously demonstrated that local replication of CD8+ T cells rather than trafficking from other sites is the most mathematically likely means of viral control (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>, <a class="xref-bibr" href="#bib26">2010</a>).</p>
                <p>Model 1 recreated a 2-cm-diameter circular area (surface area = 314 mm<sup>2</sup>) of susceptible epithelial cells in the anogenital mucosa as the typical maximum size for a single ulcer. We chose this diameter because the average surface area of recurrent genital HSV lesions (which may include many ulcers) in immunocompetent persons is 60 mm<sup>2</sup> (range 2–270 mm<sup>2</sup>) (<a class="xref-bibr" href="#bib4">Corey et al., 1983</a>), which equates to a lesion of diameter 8.7 mm (range 1.6–18.5 mm). Ulcers are rarely &gt;3 mm in diameter. Thus, we designed the model to be inclusive of the vast majority of recurrent genital ulcers in immunocompetent persons. Model 1 failed to create episodes with multiple peaks, and failed to produce episodes of &gt;4 days duration (∼33% of clinical episodes) (<a class="xref-table" href="#tbl6">Table 6</a>).</p>
            </div>
            <div id="s4-4-2" class="subsection">
                <h4>Mathematical model 2: differentiation of cell-associated and cell-free viruses</h4>
                <p>In an attempt to improve model fit, we iteratively updated the model to more realistically represent infection, according to our clinical and experimental observations of HSV-2 biology. We next constructed a model, which differentiated cell-associated from cell-free particles (Model 2) and assumed that cell-associated particles (V<sub>i</sub>) can passage from cell to cell as soon as they form within a cell: this assumption is based on studies that document transfer of HSV-2 particles via tight junctions that bind epidermal cells to one another (<a class="xref-bibr" href="#bib3">Collins and Johnson, 2003</a>). Model 1 made the assumption that all viral particles had this property because if we assumed that virus only became infectious after infected cell lysis, then rate of episode expansion was far too slow. (In fact, the majority of published virological models to date, by assuming homogeneous mixing, also assume that newly formed particles are instantly infectious, which implies cell-to-cell spread.)</p>
                <p>In Model 2, cell-associated particles (V<sub>i</sub>) converted to cell-free virus (V<sub>e</sub>) after infected cells ruptured. We assumed in Model 2, that cell-free particles were noninfectious and decayed according to a constant linear function, whereas cell-associated particles decayed according to the decay rate of infected cells, which in turn is a function of infected cell death rate and CD8+ T-cell density. Because swabs probably do not remove most cell-associated particles and infected cells (otherwise swabbing would mitigate shedding), we fit V<sub>e</sub> to the data in Cohort E. Model 2 also failed to create episodes with multiple peaks, and failed to produce episodes of &gt;4 days duration (<a class="xref-table" href="#tbl6">Table 6</a>).</p>
            </div>
            <div id="s4-4-3" class="subsection">
                <h4>Mathematical model 3: a nonspatially constrained model with numerous regions of concurrent HSV-2 replication</h4>
                <p>Based on clinical observations outlined in <a class="xref-fig" href="#fig2">Figure 2</a>, our next models were designed to test a hypothesis that multiple concurrent plaques may explain prolonged episodes due to re-expansion. Model 3 consisted of 300 possible regions of infection, linked by the ability of neurons to randomly release virus into any of the 300 regions. Moreover, we allowed formation of new plaques within other regions by cell-free particles. Cell-free particles in the model were assigned a new infectivity parameter (β<sub>e</sub>). In this model, there was no spatial constraint of virus, such that HSV from one region could initiate new sites of replication in the 299 other regions. We included parameter (ε) to account for delay in viral production within secondary plaques after seeding. This parameter was estimated based on simple experiments that we conducted with a red fluorescent protein-VP26 HSV construct that turns cells red following viral replication and encapsidation of viral DNA (VP 26 is a late capsid protein). Individual cells turn red approximately 12–24 hr after very low multiplicity of infection (MOI = 0.001) of Vero cells, which are a cell line of monkey kidney epithelial cells that are permissive for HSV entry, replication, and spread. Therefore, ε was assigned an initial value range of 0.5–1. Model 3 produced longer episodes in general than Model 2, and successfully created episodes with re-expansion. Nevertheless, episodes were too short in duration to reproduce the frequency histogram for episode duration (<a class="xref-table" href="#tbl6">Table 6</a>).</p>
            </div>
            <div id="s4-4-4" class="subsection">
                <h4>Mathematical model 4: the spatial model</h4>
                <p>Model 4 (<a class="xref-fig" href="#fig3">Figure 3</a>) is described in the ‘Results’ and <a class="xref-table" href="#tbl5">Table 5</a>. To slow spread from region to region and because HSV-2 plaques often congregate in proximity to one another, we created a spatial diagram of the genital tract and limited spread of cell-free particles from infected regions to contiguous regions only. Regions were arrayed in a two-dimensional matrix of hexagons such that cell-free particles from a region could only infect a maximum of six other regions (<a class="xref-fig" href="#fig3">Figure 3</a>), thus limiting rate of HSV-2 spread from region to region within the genital tract. For edge regions within Model 4, there were only 4 contiguous regions at risk.</p>
                <p>The regions were linked immunologically according to parameter ρ. When an ulcer was initiated in a region (infection of a single epidermal cell), the CD8+ T-cell density was assumed to include a certain proportion of CD8+ T-cell density in surrounding regions. If ρ = 0, then CD8+ T-cell density in a region was independent of surrounding regions. This implies that all immunity was localized only to an area with viral replication. If ρ = 1, CD8+ T-cell density in a region was assumed to equal the average from surrounding regions.</p>
            </div>
            <div id="s4-4-5" class="subsection">
                <h4>Parameter value search</h4>
                <p>We explored &gt;1000 parameter sets within each of our four possible models using Latin hypercube sampling. The goal was to arrive at a parameter value set that maximized the fit of each model to the shedding summary measures. Each parameter was initially assigned a wide possible range of values based on previous literature review (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>). The typical search scheme involved adjusting three randomly selected parameters at a time with five possible parameter values for each parameter leading to 5<sup>3</sup> or 125 parameter sets: the parameter set with the closest fit to model output was selected for the next round of simulations. There were a maximum of 10 unknown parameters per model so that one cycle through the entire parameter set involved 375–500 simulations (3–4 rounds of 125 simulations). After a single cycle through all parameter sets, several more cycles were performed with parameter ranges re-expanded around revised best parameter values. When there was no improvement with successive cycles through the parameter sets, we narrowed the ranges by 50% and cycled through the 375- to 500-parameter sets again. The three parameters that were concurrently varied were randomly selected with each fit. For each model, we continued this process of parameter set selection and parameter range narrowing until the fitting scores no longer improved. The best fitting score for a particular model was then used for comparison to other models.</p>
                <p>Because of the stochastic nature of our model, there was a reasonable amount of run-to-run variability, even with 10-year simulations (<a class="xref-table" href="#tbl4">Table 4</a>). We therefore repeated simulations with our spatial model that ultimately reproduced the entirety of our dataset. We arrived at similar parameter values when repeating the fitting process.</p>
                <p>Five of the 10 model parameters used in the spatial model (Model 4) have been estimated experimentally (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>). These values included β<sub>i</sub> = 6e-8 DNA copy days per cell (∼100 viruses needed per day to infect one adjacent cell) (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>), <em>p</em> = 1e5 HSV DNA particles per day (based on ∼1e3 particle forming unit [pfu] per cell and ∼100 HSV DNA copies per pfu) (<a class="xref-bibr" href="#bib24">Sacks and Schaffer, 1987</a>; <a class="xref-bibr" href="#bib14">Jiang et al., 2007a</a>; <a class="xref-bibr" href="#bib15">Jiang et al., 2007b</a>), <em>c</em> = 10 per day (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>; <a class="xref-bibr" href="#bib34">Turner et al., 1982</a>) and θ = 2.9 per day (<a class="xref-bibr" href="#bib7">De Boer et al., 2003</a>). A pfu represents a viral particle that can form a new infectious plaque in cell culture. Pfus are produced at a lower rate than HSV DNA because not all replicated viral genomes are infectious. As above, we assigned a value for ε = 0.66. We therefore attempted model fit by varying only the remaining five parameters to minimize the likelihood of multiple solutions to our model. We tested four parameter values in this case leading to 4<sup>5</sup> or 1024 parameter sets: the parameter set with the closest fit to model output was selected for the next round of simulations. When fitting with 5 rather than 10 unknown parameters, the sum of squares and AIC scores indicated a very slight decrease in fit to Cohort A data (<a class="xref-table" href="#tbl6">Table 6</a>).</p>
            </div>
            <div id="s4-4-6" class="subsection">
                <h4>Model output analysis</h4>
                <p>A feature shared by all models described in this manuscript is sensitivity to CD8+ T-cell density at episode onset, which had an important effect on amount of virus produced per episode (<a class="xref-bibr" href="#bib26">Schiffer et al., 2010</a>). We started each simulation with zero infected cells and viruses. To avoid creating a bias on model outcomes due to initial CD8+ T-cell values, we ran the model with its particular parameter set for 365 days, and then used the CD8+ T-cell values in each region at the end of these 365 days as the starting values for the recorded simulation. For analyses of shorter duration (e.g., 30- to 60-day simulations in <a class="xref-table" href="#tbl4">Table 4</a>), we ensured a random distribution of CD8+ T-cell starting values by randomly selecting a starting time between 4 and 10 years for sampling after initial simulations of the model.</p>
                <p>We calculated the reproductive number (R = [<em>p</em> × β<sub>i</sub> × S0])/([<em>a</em> + <em>f</em>E] × <em>c</em>) continually during each simulation. The reproductive number is the average number of cells that an infected cell would infect assuming the presence of CD8+ T cells. For our spatial models, which divided the genital tract into 300 separate regions that were susceptible to viral replication, we calculated R separately within each region. We calculated plaque diameter in millimeters (diameter = 8.32e-3 × √(S0 − S − I)) continually during each simulation by estimating the number of missing cells due to cell death from infection and using the diameters of these cells to estimate the size of each plaque. Our formula for plaque diameter incorporates measures of individual cell volume and ulcer depth: derivations of our estimates for R and plaque diameter are available in previous publications (<a class="xref-bibr" href="#bib25">Schiffer et al., 2009</a>, <a class="xref-bibr" href="#bib26">2010</a>). Our formula for diameter predicts that the loss of 14,400, 57,800, and 130,000 cells at any given point in time will lead to 1-, 2-, and 3-mm-diameter ulcers, respectively.</p>
                <p>Upon arriving at the most accurate model and set of parameter values, we conducted numerous simulations, which are described in the ‘Results’, to better understand the model's dynamic features. We simulated 100 episodes with at least 10,000 infected cells within a single ulcer-forming region to describe features of more severe episodes. Next, we calculated the infected cell growth rate (<em>g</em>) from 1 to peak cells for each episode (<em>g</em> = (ln I<sub>peak</sub>)/<em>t</em><sub>peak</sub>), and used this measure to roughly estimate generation time (Tg, time from becoming infected to infecting a neighboring cell for an infected cell), where T<sub>g</sub> = ((R<sub>init</sub> − 1)/2))/<em>g</em> (<a class="xref-bibr" href="#bib10">Fraser et al., 2004</a>). The reproductive number was halved in this equation because the number transitions from R<sub>init</sub> to a value of 1 at episode peak. We similarly calculated infected cell doubling time for each episode according to: ln (2) T<sub>g</sub>/((R<sub>init</sub> − 1)/2).</p>
            </div>
            <div id="s4-4-7" class="subsection">
                <h4>Sensitivity analyses</h4>
                <p>Our local sensitivity analysis was based on conducting prolonged simulations with single parameters adjusted until fit to the dataset was no longer optimal. We also performed global sensitivity analyses with Monte Carlo simulations to better understand critical dynamics of the system.</p>
                <p>For global sensitivity analyses, all model parameters were randomly drawn from probability distribution functions (pdfs), which were derived according to ranges of parameters from model fit (<a class="xref-table" href="#tbl2">Table 2</a>). We first simulated 500 episodes each of which was assigned a unique parameter set. Region and timing of episode initiation was selected stochastically to allow a range of initial CD8+ T-cell densities. Partial rank correlation coefficients (PRCCs) (<a class="xref-bibr" href="#bib17">Marino et al., 2008</a>) and Akaike Information Criteria (AIC) were used to identify important predictive parameters of episode duration and peak viral load. We next performed 500 10-year simulations each associated with a unique parameter set, and examined the effect of parameter variability on shedding frequency and episode rate.</p>
            </div>
        </div>
    </div>
</div>